# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# Arpad Biro <biro_arpad@yahoo.com>, 2004, 2005, 2006, 2007.
# Gabor SUVEG <gabor@suveg.hu>, 2004.
# Gábor Szentiványi <szenti@ulx.hu>, 2006.
# Hoppár Zoltán <hopparz@gmail.com>, 2009.
# Sulyok Péter <peti@fedoraproject.org>, 2008, 2009.
# Széll Tamás <tomi@digiflex.hu>, 2004.
# Szentiványi Gábor <szentivanyi.gabor@ulx.hu>, 2006.
msgid ""
msgstr ""
"Project-Id-Version: Anaconda\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2013-01-22 17:22-0800\n"
"PO-Revision-Date: 2013-01-08 18:22+0000\n"
"Last-Translator: Brian Lane <bcl@redhat.com>\n"
"Language-Team: Hungarian <trans-hu@lists.fedoraproject.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: hu\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: anaconda:366
msgid "Press <enter> for a shell"
msgstr "Parancsértelmezőért nyomjon <Enter>-t"

#: anaconda:392 rescue.py:353 rescue.py:379 rescue.py:392 rescue.py:470
#: rescue.py:483 rescue.py:494 text.py:663 loader/windows.h:31
#: loader/cdinstall.c:209 loader/cdinstall.c:213 loader/cdinstall.c:240
#: loader/cdinstall.c:244 loader/cdinstall.c:260 loader/cdinstall.c:263
#: loader/cdinstall.c:432 loader/cdinstall.c:439 loader/cdinstall.c:512
#: loader/dirbrowser.c:128 loader/driverdisk.c:294 loader/driverdisk.c:478
#: loader/driverdisk.c:514 loader/driverdisk.c:545 loader/driverdisk.c:586
#: loader/driverdisk.c:602 loader/driverdisk.c:614 loader/driverdisk.c:622
#: loader/driverdisk.c:802 loader/driverdisk.c:840 loader/driverselect.c:78
#: loader/driverselect.c:178 loader/hdinstall.c:117 loader/hdinstall.c:268
#: loader/hdinstall.c:327 loader/hdinstall.c:362 loader/hdinstall.c:442
#: loader/hdinstall.c:492 loader/hdinstall.c:505 loader/kbd.c:119
#: loader/kickstart.c:138 loader/kickstart.c:148 loader/kickstart.c:190
#: loader/kickstart.c:195 loader/kickstart.c:302 loader/kickstart.c:334
#: loader/kickstart.c:524 loader/lang.c:114 loader/lang.c:373
#: loader/loader.c:443 loader/loader.c:479 loader/loader.c:519
#: loader/loader.c:537 loader/loader.c:554 loader/loader.c:591
#: loader/loader.c:1252 loader/loader.c:1281 loader/loader.c:1496
#: loader/mediacheck.c:46 loader/mediacheck.c:85 loader/mediacheck.c:92
#: loader/mediacheck.c:96 loader/mediacheck.c:105 loader/method.c:277
#: loader/method.c:351 loader/modules.c:396 loader/modules.c:412
#: loader/net.c:607 loader/net.c:991 loader/net.c:1796 loader/net.c:1820
#: loader/net.c:2164 loader/net.c:2183 loader/net.c:2195
#: loader/nfsinstall.c:91 loader/nfsinstall.c:279 loader/nfsinstall.c:296
#: loader/nfsinstall.c:384 loader/telnetd.c:92 loader/urlinstall.c:137
#: loader/urlinstall.c:159 loader/urlinstall.c:221 loader/urlinstall.c:374
#: loader/urlinstall.c:385 loader/urlinstall.c:392 loader/urls.c:268
#: loader/urls.c:339 loader/urls.c:345 loader/urls.c:354
#: textw/constants_text.py:44
msgid "OK"
msgstr "OK"

#: anaconda:401
msgid ""
"You do not have enough RAM to use the graphical installer.  Starting text "
"mode."
msgstr ""
"Nincs elég memória a grafikus telepítő használatához.  Telepítés szöveges "
"módban."

#: anaconda:535
msgid "Starting graphical installation."
msgstr "Grafikus telepítő indul…"

#: anaconda:911
msgid "Would you like to use VNC?"
msgstr "Szeretne VNC-t használni?"

#: anaconda:912
msgid ""
"Text mode provides a limited set of installation options.  It does not allow "
"you to specify your own partitioning layout or package selections.  Would "
"you like to use VNC mode instead?"
msgstr ""
"A szöveges mód lekorlátozott telepítés opciókat kínál.  Nem engedi meg a "
"saját merevlemez felosztást vagy a csomagok kiválogatását.  Kívánja inkább a "
"VNC módot használni?"

#: anaconda:940
msgid "Install class forcing text mode installation"
msgstr "A telepítő osztály szöveges módú telepítést erőltet"

#: anaconda:959
msgid "Graphical installation is not available. Starting text mode."
msgstr "Grafikus telepítés nem érhető el. Szöveges mód indítása."

#: anaconda:966
msgid "DISPLAY variable not set. Starting text mode."
msgstr "DISPLAY változó nincs beállítva. Szöveges mód indítása."

#: backend.py:154
#, python-format
msgid "Upgrading %s\n"
msgstr "%s frissítése\n"

#: backend.py:156
#, python-format
msgid "Installing %s\n"
msgstr "%s telepítése\n"

#: bootloader.py:55 bootloader.py:242 bootloader.py:248 bootloader.py:255
#: gui.py:1284 image.py:82 installinterfacebase.py:66 network.py:880
#: text.py:490 text.py:550 yuminstall.py:1621 yuminstall.py:1830
#: yuminstall.py:1865 iw/blpasswidget.py:154 iw/upgrade_swap_gui.py:190
#: iw/upgrade_swap_gui.py:198 iw/upgrade_swap_gui.py:205
#: textw/upgrade_text.py:196
msgid "Warning"
msgstr "Figyelmeztetés"

#: bootloader.py:56
msgid ""
"Filesystems have already been activated.  You cannot go back past this "
"point.\n"
"\n"
"Would you like to continue with the installation?"
msgstr ""
"A fájlrendszerek már működnek.  Ettől a ponttól folytatva, már nincs "
"visszalépés.\n"
"\n"
"Szeretné folytatni a telepítéssel?"

#: bootloader.py:60 gui.py:1187 gui.py:1320 gui.py:1539 image.py:91
#: installinterfacebase.py:109 livecd.py:205 livecd.py:447 packages.py:162
#: upgrade.py:58 upgrade.py:199 yuminstall.py:272 yuminstall.py:1099
#: yuminstall.py:1104 yuminstall.py:1187 yuminstall.py:1193 yuminstall.py:1345
#: yuminstall.py:1372 yuminstall.py:1433 yuminstall.py:1617 yuminstall.py:1647
#: yuminstall.py:1670 storage/__init__.py:105 storage/__init__.py:111
#: storage/__init__.py:113 storage/__init__.py:221 storage/__init__.py:274
#: storage/__init__.py:2087
msgid "_Exit installer"
msgstr "_Kilépés a telepítőből"

#: bootloader.py:60 image.py:92 image.py:253 kickstart.py:1506
#: kickstart.py:1550 upgrade.py:58 yuminstall.py:1438 yuminstall.py:1617
#: iw/partition_gui.py:1629 storage/__init__.py:2088
msgid "_Continue"
msgstr "_Tovább"

#: bootloader.py:178
msgid "Bootloader"
msgstr "Rendszerindító"

#: bootloader.py:178
msgid "Installing bootloader."
msgstr "Rendszerindító telepítése."

#: bootloader.py:243
msgid ""
"There was an error installing the bootloader.  The system may not be "
"bootable."
msgstr ""
"Hiba történt a rendszerindító telepítése közben.  A rendszer valószínűleg "
"nem lesz indítható."

#: bootloader.py:249
msgid ""
"No kernel packages were installed on the system.  Bootloader configuration "
"will not be changed."
msgstr ""
"Nem települtek rendszermag csomagok. A rendszerindító beállításai nem "
"változnak meg."

#: bootloader.py:256
msgid ""
"There was an error installing the bootloader.  The system may not be bootable"
msgstr ""

#: cmdline.py:56
msgid "Completed"
msgstr "Kész"

#: cmdline.py:64
msgid "In progress"
msgstr "Folyamatban"

#: cmdline.py:88 gui.py:1315 kickstart.py:1288 kickstart.py:1368
#: kickstart.py:1376 kickstart.py:1415 kickstart.py:1423 text.py:396
#, python-format
msgid ""
"The following error was found while parsing the kickstart configuration "
"file:\n"
"\n"
"%s"
msgstr ""
"A következő hiba merült fel a kickstart konfigurációs fájl értelmezése "
"közben:\n"
"\n"
"%s"

#: cmdline.py:100
msgid ""
"Command line mode requires all choices to be specified in a kickstart "
"configuration file."
msgstr ""
"A parancssoros módhoz minden választást meg kell adni a kickstart "
"konfigurációs fájlban."

#: cmdline.py:119 cmdline.py:126 cmdline.py:133 cmdline.py:143 cmdline.py:153
#: cmdline.py:160
msgid "Can't have a question in command line mode!"
msgstr "Nem lehet kérdezni parancssoros módban!"

#: constants.py:73
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report"
msgstr ""
"Kezeletlen kivétel történt.  A legvalószínűbb, hogy ez programhiba.  Kérem, "
"másolja ki a kivétel teljes szövegét, és adjon fel egy hibabejelentést"

#: constants.py:79
msgid " with the provider of this software."
msgstr " e szoftver szállítójának."

#: constants.py:83
#, python-format
msgid " against anaconda at %s"
msgstr " anaconda tárggyal %s helyen"

#: gui.py:107
msgid "An error occurred saving screenshots to disk."
msgstr "A képernyőképek lemezre mentése közben hiba történt."

#: gui.py:124
msgid "Screenshots Copied"
msgstr "Képernyőképek átmásolva"

#: gui.py:125
msgid ""
"The screenshots have been saved in the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""
"A képernyőképek a következő mappába kerültek:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"Újraindítás után rendszergazdaként belépve érheti el."

#: gui.py:164
msgid "Saving Screenshot"
msgstr "Képernyőkép mentése"

#: gui.py:165
#, python-format
msgid "A screenshot named '%s' has been saved."
msgstr "„%s” nevű képernyőkép elmentve."

#: gui.py:168
msgid "Error Saving Screenshot"
msgstr "A képernyőkép mentése közben hiba történt"

#: gui.py:169
msgid ""
"An error occurred while saving the screenshot.  If this occurred during "
"package installation, you may need to try several times for it to succeed."
msgstr ""
"Képernyőkép mentése közben hiba történt.  Ha ez a csomag telepítése alatt "
"történt, akkor lehet, hogy többször is meg kell próbálni, hogy sikeres "
"legyen."

#: gui.py:502
msgid "Installation Key"
msgstr "Telepítő kulcs"

#: gui.py:573 text.py:149
msgid ""
"Choose a passphrase for the encrypted devices. You will be prompted for this "
"passphrase during system boot."
msgstr ""
"Válasszon jelszót a titkosított eszközöknek. A rendszerindító induláskor "
"kérni fogja ezt a jelszót."

#: gui.py:592 gui.py:600 text.py:187 text.py:197
msgid "Error with passphrase"
msgstr "Jelszóhiba"

#: gui.py:593 text.py:188
msgid "The passphrases you entered were different.  Please try again."
msgstr "A megadott jelszók különböznek.  Kérem próbálja újra."

#: gui.py:601
msgid "The passphrase must be at least eight characters long."
msgstr "A jelszó legalább 8 jel hosszú kell legyen."

#: gui.py:635 text.py:225
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""
"%s eszköz titkosítva. Ahhoz, hogy hozzáférjek az eszköz tartalmához "
"telepítés közben, önnek alább meg kell adnia az eszköz jelszavát."

#: gui.py:726 gui.py:1539 partIntfHelpers.py:160 partIntfHelpers.py:345
#: text.py:100 text.py:101 iw/account_gui.py:131 loader/dirbrowser.c:128
#: loader/driverdisk.c:479 loader/kickstart.c:334 loader/loader.c:479
#: loader/loader.c:591 textw/constants_text.py:48 textw/userauth_text.py:87
msgid "Cancel"
msgstr "Mégsem"

#: gui.py:738 gui.py:739 gui.py:835 gui.py:907 gui.py:908
#: ui/anaconda.glade.h:3
msgid "_Debug"
msgstr "Ny_omkövetés"

#: gui.py:1042 textw/netconfig_text.py:399
msgid "Waiting for NetworkManager"
msgstr "Várakozás a NetworkManager-re"

#: gui.py:1043
#, python-format
msgid "Waiting for NetworkManager to activate these devices: %s"
msgstr ""

#: gui.py:1064
#, python-format
msgid "Failed to activate these network interfaces: %s"
msgstr ""

#: gui.py:1067
msgid "Network Configuration"
msgstr ""

#: gui.py:1076
#, python-format
msgid "An error occurred trying to bring up the %s network interface."
msgstr "%s hálózati csatolót bekapcsolván hiba történt."

#: gui.py:1078
msgid "Error Enabling Network"
msgstr ""

#: gui.py:1137 installinterfacebase.py:48 iw/task_gui.py:324
msgid "Error Setting Up Repository"
msgstr "Hiba a tároló beállítása közben"

#: gui.py:1138
#, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%(e)s\n"
"\n"
"Please provide the correct information for installing %(productName)s."
msgstr ""

#: gui.py:1236 ui/reinitialize-dialog.glade.h:2
msgid "Storage Device Warning"
msgstr ""

#: gui.py:1285 text.py:551
#, python-format
msgid ""
"Error processing LVM.\n"
"There is inconsistent LVM data on %(msg)s.  You can reinitialize all related "
"PVs (%(pvs)s) which will erase the LVM metadata, or ignore which will "
"preserve the contents.  This action may also be applied to all other PVs "
"with inconsistent metadata."
msgstr ""

#: gui.py:1292 installinterfacebase.py:94 text.py:500 text.py:558
msgid "_Ignore"
msgstr "_Mellőz"

#: gui.py:1293 text.py:501 text.py:559
msgid "Ignore _all"
msgstr ""

#: gui.py:1294 text.py:502 text.py:560
msgid "_Re-initialize"
msgstr "Újra _alakít"

#: gui.py:1295 text.py:503 text.py:561
msgid "Re-ini_tialize all"
msgstr ""

#: gui.py:1317 text.py:398
msgid "Error Parsing Kickstart Config"
msgstr "Hiba történt a Kickstart beállítás feldolgozásakor"

#: gui.py:1364
msgid "default:LTR"
msgstr "alapértelmezés:LTR"

#: gui.py:1444 text.py:628
msgid "Error!"
msgstr "Hiba!"

#: gui.py:1445 text.py:629
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""
"Telepítő felülelet komponensét betöltvén hiba történt.\n"
"\n"
"osztály = %s"

#: gui.py:1450 image.py:166 image.py:196 packages.py:370
#: storage/__init__.py:1905
msgid "_Exit"
msgstr "_Kilépés"

#: gui.py:1451 image.py:166 image.py:196 livecd.py:205 yuminstall.py:966
#: yuminstall.py:1372 yuminstall.py:1433 yuminstall.py:1647
msgid "_Retry"
msgstr "Újr_a"

#: gui.py:1453 storage/partitioning.py:313
msgid "The system will now reboot."
msgstr "A rendszer most újraindul."

#: gui.py:1454 image.py:253 packages.py:374 yuminstall.py:1688
msgid "_Reboot"
msgstr "Újrai_ndítás"

#: gui.py:1456
msgid "Exiting"
msgstr "Kilépés"

#: gui.py:1536 installinterfacebase.py:55 livecd.py:126 text.py:267
#: upgrade.py:190
msgid "Exit installer"
msgstr "Kilépés a telepítőből"

#: gui.py:1537
msgid "Are you sure you wish to exit the installer?"
msgstr "Biztos, hogy ki kíván lépni a telepítőből?"

#: gui.py:1546
#, python-format
msgid "%s Installer"
msgstr "%s telepítő"

#: gui.py:1552
msgid "Unable to load title bar"
msgstr "Nem sikerült betölteni a címsort"

#: gui.py:1635
msgid "Install Window"
msgstr "Telepítőablak"

#: image.py:83
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error."
msgstr ""
"%s ISO képmás mérete nem 2048 bájt többszöröse.  Ez azt is jelentheti, hogy "
"erre a számítógépre áthozva megsérült a képmás.\n"
"\n"
"Ajánlom, hogy lépjen ki, és szakítsa meg a telepítést. Azonban választhatja "
"a folytatást is, ha úgy gondolja, hogy ez egy hiba."

#: image.py:158
msgid "Couldn't Mount ISO Source"
msgstr "Nem sikerült ISO forrást becsatolni"

#: image.py:159
#, python-format
msgid ""
"An error occurred mounting the source device %s.  This may happen if your "
"ISO images are located on an advanced storage device like LVM or RAID, or if "
"there was a problem mounting a partition.  Click exit to abort the "
"installation."
msgstr ""
"%s forráseszköz becsatolása közben hiba történt.  Ez megtörténhet, ha az ISO "
"képmások egy fejlett tároló eszközön vannak (mint LVM vagy RAID), vagy ha "
"baj volt a lemezrész becsatolásával.  Kattintson a kilépésre, hogy kilépjen "
"a telepítőből."

#: image.py:187
msgid "Missing ISO 9660 Image"
msgstr "Hiányzó ISO 9660 képmás"

#: image.py:188
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the hard "
"drive.\n"
"\n"
"Please copy this image to the drive and click Retry.  Click Exit to abort "
"the installation."
msgstr ""
"A telepítő megpróbálta becsatolni #%s képet, de nem találja a merevlemezen.\n"
"\n"
"Kérem másolja ezt a képet a merevlemezre, és kattintson az Újra gombra.  "
"Kattintson a Kilépésre, hogy megszakítsa a telepítést."

#: image.py:243
msgid "Required Install Media"
msgstr "Szükséges telepítő adathordozók"

#: image.py:244
#, python-format
msgid ""
"The software you have selected to install will require the following %"
"(productName)s %(productVersion)s discs:\n"
"\n"
"%(reqcdstr)s\n"
"Please have these ready before proceeding with the installation.  If you "
"need to abort the installation and exit please select \"Reboot\"."
msgstr ""

#: image.py:253 livecd.py:446 packages.py:374 upgrade.py:198
#: yuminstall.py:1104 yuminstall.py:1193 yuminstall.py:1619 yuminstall.py:1670
#: yuminstall.py:1688 storage/__init__.py:113 ui/anaconda.glade.h:2
msgid "_Back"
msgstr "_Vissza"

#: image.py:291 livecd.py:198 livecd.py:440 partIntfHelpers.py:240 text.py:359
#: text.py:363 yuminstall.py:446 yuminstall.py:982 yuminstall.py:1097
#: yuminstall.py:1102 yuminstall.py:1376 yuminstall.py:1443 yuminstall.py:1641
#: yuminstall.py:1663 iw/advanced_storage.py:385 iw/advanced_storage.py:398
#: iw/advanced_storage.py:458 iw/autopart_type.py:95 iw/cleardisks_gui.py:44
#: iw/cleardisks_gui.py:54 iw/filter_gui.py:437 iw/osbootwidget.py:211
#: iw/osbootwidget.py:221 iw/raid_dialog_gui.py:220 iw/task_gui.py:54
#: iw/task_gui.py:152 iw/task_gui.py:305 iw/task_gui.py:472
#: loader/windows.h:31 loader/cdinstall.c:240 loader/cdinstall.c:512
#: loader/driverdisk.c:294 loader/driverdisk.c:545 loader/driverdisk.c:586
#: loader/driverdisk.c:614 loader/driverdisk.c:622 loader/driverdisk.c:694
#: loader/hdinstall.c:117 loader/hdinstall.c:327 loader/hdinstall.c:442
#: loader/hdinstall.c:492 loader/hdinstall.c:505 loader/kickstart.c:302
#: loader/lang.c:114 loader/loader.c:443 loader/loader.c:554
#: loader/loader.c:1252 loader/mediacheck.c:46 loader/mediacheck.c:85
#: loader/mediacheck.c:92 loader/mediacheck.c:96 loader/method.c:277
#: loader/method.c:351 loader/nfsinstall.c:279 loader/nfsinstall.c:296
#: loader/telnetd.c:92 loader/urlinstall.c:137 loader/urlinstall.c:159
#: loader/urlinstall.c:221 loader/urls.c:339 loader/urls.c:345
#: loader/urls.c:354 storage/__init__.py:270 storage/__init__.py:1906
#: storage/__init__.py:1993 storage/zfcp.py:375 textw/upgrade_text.py:184
#: textw/upgrade_text.py:191
msgid "Error"
msgstr "Hiba"

#: image.py:292
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""
"Lemez lecsatolása közben hiba történt.  Kérem győződjön meg arról, hogy nem "
"használja %s eszközt egy parancsértelmezőből a tty2-n, majd kattintson az OK "
"gombra, hogy újra megpróbálja."

#: installclass.py:71
msgid "Install on System"
msgstr "Telepítés a rendszerre"

#: installinterfacebase.py:49
#, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%(e)s\n"
"\n"
"Installation can not continue."
msgstr ""

#: installinterfacebase.py:67
#, python-format
msgid ""
"Disk %s contains BIOS RAID metadata, but is not part of any recognized BIOS "
"RAID sets. Ignoring disk %s."
msgid_plural ""
"Disks %s contain BIOS RAID metadata, but are not part of any recognized BIOS "
"RAID sets. Ignoring disks %s."
msgstr[0] ""
msgstr[1] ""

#: installinterfacebase.py:80
msgid "Unformatted DASD Device Found"
msgid_plural "Unformatted DASD Devices Found"
msgstr[0] ""
msgstr[1] ""

#: installinterfacebase.py:82
#, python-format
msgid ""
"Format uninitialized DASD device?\n"
"\n"
"There is %d uninitialized DASD device on this system.  To continue "
"installation, the device must be formatted.  Formatting will remove any data "
"on this device."
msgid_plural ""
"Format uninitialized DASD devices?\n"
"\n"
"There are %d uninitialized DASD devices on this system.  To continue "
"installation, the devices must be formatted.  Formatting will remove any "
"data on these devices."
msgstr[0] ""
msgstr[1] ""

#: installinterfacebase.py:94 iw/partition_gui.py:900
msgid "_Format"
msgstr "_Formázás"

#: installinterfacebase.py:102
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your "
"hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""

#: installinterfacebase.py:105
msgid "Hardware Error Encountered"
msgstr ""

#: iutil.py:853
msgid "Error determining boot device's disk name"
msgstr ""

#: iutil.py:876
msgid "the device containing /boot"
msgstr ""

#: iutil.py:878
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from %s to continue installation."
msgstr ""

#: kickstart.py:121
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  You may "
"examine the output in %(msgs)s.  This is a fatal error and installation will "
"be aborted.  Press the OK button to exit the installer."
msgstr ""

#: kickstart.py:129 kickstart.py:131
msgid "Scriptlet Failure"
msgstr "Kickstart szkript hiba"

#: kickstart.py:155
#, python-format
msgid ""
"Escrow certificate with url %s requires network to be enabled in loader or "
"configured in kickstart file."
msgstr ""

#: kickstart.py:170
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""

#: kickstart.py:1387
#, python-format
msgid "Error processing %%ksappend lines: %s"
msgstr "Hiba %%ksappend sorok feldolgozása alatt: %s"

#: kickstart.py:1390
#, python-format
msgid "Unknown error processing %%ksappend lines: %s"
msgstr "Ismeretlen hiba %%ksappend sorok feldolgozása alatt: %s"

#: kickstart.py:1446 livecd.py:228
msgid "Post-Installation"
msgstr "Zárótelepítés"

#: kickstart.py:1447
msgid "Running post-installation scripts"
msgstr "Telepítést záró szkriptek futtatása"

#: kickstart.py:1463
msgid "Pre-Installation"
msgstr "Telepítés előkészítés"

#: kickstart.py:1464
msgid "Running pre-installation scripts"
msgstr "Telepítés előkészítési szkriptek futtatása"

#: kickstart.py:1497
msgid "Missing Package"
msgstr "Hiányzó csomag"

#: kickstart.py:1498
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist. Would you like to continue or abort this installation?"
msgstr ""
"„%s” csomagot megjelölte telepítésre.  Nincs ilyen csomag. Folytatni akarja, "
"vagy megszakítja a telepítést?"

#: kickstart.py:1504 kickstart.py:1548
msgid "_Abort"
msgstr "_Megszakítás"

#: kickstart.py:1505 kickstart.py:1549
msgid "_Ignore All"
msgstr "_Mindegyik mellőzése"

#: kickstart.py:1540
msgid "Missing Group"
msgstr "Hiányzó csoport"

#: kickstart.py:1541
#, python-format
msgid ""
"You have specified that the group '%s' should be installed. This group does "
"not exist. Would you like to continue or abort this installation?"
msgstr ""
"„%s” csoportot megjelölte telepítésre. Nincs ilyen csoport. Folytatni "
"akarja, vagy megszakítja a telepítést?"

#: kickstart.py:1664
#, python-format
msgid ""
"The kickstart configuration file is missing required information that "
"anaconda cannot prompt for.  Please add the following sections and try "
"again:\n"
"%s"
msgstr ""
"A kickstart beállítás fájlból olyan adat hiányzik, amit a telepítő nem tud "
"bekérni.  Kérem adja hozzá a következő szakaszokat, majd próbálja újra:\n"
"%s"

#: livecd.py:121
msgid "Unable to find image"
msgstr "Képmás nem található"

#: livecd.py:122
#, python-format
msgid ""
"The given location isn't a valid %s live CD to use as an installation source."
msgstr ""
"A megadott hely nem érvényes telepítési forrásként használható %s élő CD."

#: livecd.py:180
msgid "Copying live image to hard drive."
msgstr "Élő képmás másolása a merevlemezre."

#: livecd.py:199
msgid ""
"There was an error installing the live image to your hard drive.  This could "
"be due to bad media.  Please verify your installation media.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"require reinstallation."
msgstr ""
"Hiba történt az élő kép merevlemezre másolása közben.  Talán sérült hordozó "
"okozta.  Kérem ellenőrizze a telepítő hordozót.\n"
"\n"
"Ha kilép, a rendszer töredékkel teli marad, s emiatt újbóli telepítést "
"igényel."

#: livecd.py:229
msgid ""
"Performing post-installation filesystem changes.  This may take several "
"minutes."
msgstr ""
"Fájlrendszer változásainak zárótelepítése.  Ez néhány percig eltarthat."

#: livecd.py:441
#, python-format
msgid ""
"The root filesystem you created is not large enough for this live image "
"(%.2f MB required)."
msgstr ""
"A gyökér fájlrendszer, amit ön készített nem elég nagy ehhez az élő képhez "
"(%.2f·MB·szükséges)."

#: network.py:72
msgid "Hostname must be 255 or fewer characters in length."
msgstr "A gazdanév legfeljebb 255 jelből állhat."

#: network.py:78
msgid ""
"Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or '0-"
"9'"
msgstr ""
"A gazda név első betűje az „a-z”, „A-Z” vagy „0-9” intervallumban kell "
"legyen."

#: network.py:83
msgid ""
"Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.'"
msgstr ""
"A gazda nevében kizárólag engedélyezett jelek: 'a-z', 'A-Z', '0-9', '-', "
"vagy '.'"

#: network.py:127
msgid "IP address is missing."
msgstr "IP cím hiányzik."

#: network.py:131
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr ""
"IPv4 cím 4 darab, 0 és 255 közé eső, egymástól ponttal elválasztott számot "
"kell tartalmazzon."

#: network.py:134
#, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr "„%s” nem érvényes IPv6 cím."

#: network.py:136
#, python-format
msgid "'%s' is an invalid IP address."
msgstr "„%s” érvénytelen IP cím."

#: network.py:881
msgid ""
"You do not have an active network connection.  This is required by some "
"exception saving methods.  Would you like to configure your network now?"
msgstr ""

#: network.py:887 rescue.py:262 yuminstall.py:1340 iw/task_gui.py:311
msgid "No Network Available"
msgstr "Nincs elérhető hálózat"

#: network.py:888
msgid "Remote exception saving methods will not work."
msgstr ""

#: packages.py:114
msgid "Device Resize Failed"
msgstr "Az eszköz átméretezése sikertelen"

#: packages.py:115
#, python-format
msgid "An error was encountered while resizing device %s."
msgstr "%s eszköz átméretezése alatt hiba történt."

#: packages.py:119
msgid "Device Creation Failed"
msgstr "Eszköz elkészítése sikertelen"

#: packages.py:120
#, python-format
msgid "An error was encountered while creating device %s."
msgstr "%s eszköz elkészítése közben hiba történt."

#: packages.py:124
msgid "Device Removal Failed"
msgstr "Eszköz eltávolítása sikertelen"

#: packages.py:125
#, python-format
msgid "An error was encountered while removing device %s."
msgstr "%s eszköz eltávolítása közben hiba történt."

#: packages.py:129
msgid "Device Setup Failed"
msgstr "Eszköz beállítása sikertelen"

#: packages.py:130
#, python-format
msgid "An error was encountered while setting up device %s."
msgstr "%s eszköz beállítása közben hiba történt."

#: packages.py:134
msgid "Resizing Failed"
msgstr "Átméretezés sikertelen"

#: packages.py:135
#, python-format
msgid "There was an error encountered while resizing the device %s."
msgstr "%s eszköz átméretezése közben hiba történt."

#: packages.py:143
msgid "Migration Failed"
msgstr "A migráció sikertelen"

#: packages.py:144
#, python-format
msgid "An error was encountered while migrating filesystem on device %s."
msgstr "A fájlrendszert átmozgatása közben hiba történt a %s eszközön."

#: packages.py:148
msgid "Formatting Failed"
msgstr "A formázás sikertelen"

#: packages.py:149
#, python-format
msgid "An error was encountered while formatting device %s."
msgstr "%s eszköz formázása közben hiba lépett fel."

#: packages.py:154
msgid "Storage Activation Failed"
msgstr "Tárolóeszköz aktiválása sikertelen"

#: packages.py:155
msgid "An error was encountered while activating your storage configuration."
msgstr "A tároló beállításainak érvényesítése közben hiba lépett fel."

#: packages.py:162
msgid "_File Bug"
msgstr "_Hiba iktatása"

#: packages.py:354 packages.py:375
msgid "Warning! This is pre-release software!"
msgstr "Figyelmeztetés! Ez kiadás előtti szoftver!"

#: packages.py:355
#, python-format
msgid ""
"Thank you for downloading this pre-release of %(productName)s.\n"
"\n"
"This is not a final release and is not intended for use on production "
"systems.  The purpose of this release is to collect feedback from testers, "
"and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %(bugzillaUrl)s\n"
"\n"
"and file a report against '%(fileagainst)s'.\n"
msgstr ""

#: packages.py:370
msgid "_Install anyway"
msgstr "_Telepítés mindenképpen"

#: packages.py:373
msgid "Your system will now be rebooted..."
msgstr "A rendszer újraindítása következik..."

#: partIntfHelpers.py:48
msgid "Please enter a volume group name."
msgstr "Kérem adjon meg egy kötetcsoportnevet."

#: partIntfHelpers.py:52
#, python-format
msgid "Volume Group Names must be less than %d characters"
msgstr ""

#: partIntfHelpers.py:55
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr "Hiba történt – %s kötet csoport név érvénytelen."

#: partIntfHelpers.py:60
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"Hiba – a kötet csoport név érvénytelen jeleket vagy szóközöket tartalmaz. "
"Csak ékezetlen betű, számjegy, „.” és „_” használható."

#: partIntfHelpers.py:70
msgid "Please enter a logical volume name."
msgstr "Kérem adjon meg logikai kötet nevet."

#: partIntfHelpers.py:74
#, python-format
msgid "Logical Volume Names must be less than %d characters"
msgstr ""

#: partIntfHelpers.py:78
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr "Hiba – %s logikai kötet név érvénytelen."

#: partIntfHelpers.py:84
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"Hiba – a logikai kötet név érvénytelen jelet vagy szóközt tartalmaz.  Csak "
"ékezetlen betű, számjegy, „.” vagy „_” használható."

#: partIntfHelpers.py:112
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""
"%s csatoló pont érvénytelen.  Csatoló pontnak „/” jellel kell kezdődnie, nem "
"végződhet „/” jelre, és csak nyomtatható jeleket tartalmazhat, szóközöket "
"nem."

#: partIntfHelpers.py:124 partIntfHelpers.py:131
msgid "Unable To Delete"
msgstr "Nem sikerült törölni"

#: partIntfHelpers.py:125
msgid "You must first select a partition to delete."
msgstr "Először ki kell választani a törölni kívánt partíciót."

#: partIntfHelpers.py:156 partIntfHelpers.py:344 iw/lvm_dialog_gui.py:887
msgid "Confirm Delete"
msgstr "Törlés megerősítése"

#: partIntfHelpers.py:157
#, python-format
msgid "You are about to delete all partitions on the device '%s'."
msgstr "„%s” eszközön minden részt törölni fog."

#: partIntfHelpers.py:160 partIntfHelpers.py:345 iw/lvm_dialog_gui.py:890
#: iw/lvm_dialog_gui.py:1510 iw/osbootwidget.py:104 iw/partition_gui.py:1860
#: iw/partition_gui.py:1877
msgid "_Delete"
msgstr "Tö_rlés"

#: partIntfHelpers.py:207
msgid "Notice"
msgstr "Megjegyzés"

#: partIntfHelpers.py:208
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""
"A következő részeket nem töröltem, mert használják:\n"
"\n"
"%s"

#: partIntfHelpers.py:223
msgid "Format as Swap?"
msgstr "Kialakítás lapozóként?"

#: partIntfHelpers.py:224
#, python-format
msgid ""
"%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""
"%s partíció típusa 0x82 (Linux lapozó), de úgy tűnik, nem aszerint "
"alakították ki.\n"
"\n"
"Szeretné ezt a partíciót lapozó részként kialakítani?"

#: partIntfHelpers.py:239
#, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr "Legalább egy merevlemezt ki kell választani %s telepítéséhez."

#: partIntfHelpers.py:244
msgid ""
"You have chosen to use a pre-existing partition for this installation "
"without formatting it. We recommend that you format this partition to make "
"sure files from a previous operating system installation do not cause "
"problems with this installation of Linux. However, if this partition "
"contains files that you need to keep, such as home directories, then "
"continue without formatting this partition."
msgstr ""
"Egy már meglévő részt választott ki telepítésre megújítás nélkül. Ajánljuk, "
"hogy új fájlrendszert alakítson ki e részen, hogy az új rendszer működésében "
"biztosan ne okozzanak bajt a korábbi rendszerhez tartozó fájlok. Azonban, ha "
"ezen a részen olyan fájlok vannak, amelyeket meg szeretne őrizni, például a "
"használók saját könyvtárai, akkor folytassa megújítás nélkül."

#: partIntfHelpers.py:252
msgid "Format?"
msgstr "Formázás?"

#: partIntfHelpers.py:252 iw/partition_gui.py:1627
msgid "_Modify Partition"
msgstr "Partíció _módosítása"

#: partIntfHelpers.py:252
msgid "Do _Not Format"
msgstr "Átalakítás _nélkül"

#: partIntfHelpers.py:260
msgid "Error with Partitioning"
msgstr "Hiba a particionálásnál"

#: partIntfHelpers.py:261
#, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of %"
"(productName)s.\n"
"\n"
"%(errorstr)s"
msgstr ""

#: partIntfHelpers.py:277
msgid "Partitioning Warning"
msgstr "Particionálási figyelmeztetés"

#: partIntfHelpers.py:278
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""
"A következő figyelmeztetések keletkeztek a kért felosztási sémához.\n"
"\n"
"%s\n"
"\n"
"Szeretné folytatni a kért felosztó sémával?"

#: partIntfHelpers.py:292
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""
"A következő korábban már kialakított partíciókat jelölte ki formázásra az "
"összes adat törlésével."

#: partIntfHelpers.py:295
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""
"Válassza az „Igen”-t a folytatáshoz és a partíciók formázásához, vagy a "
"„Nem”-et a visszalépéshez és a beállítások megváltoztatásához."

#: partIntfHelpers.py:301
msgid "Format Warning"
msgstr "Formázási figyelmeztetés"

#: partIntfHelpers.py:328
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""
"„%s” kötet csoport törlésére készül.\n"
"\n"
"A kötet csoporthoz tartozó összes logikai kötet EL FOG VESZNI!"

#: partIntfHelpers.py:332
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr "„%s” logikai kötet törlésére készül."

#: partIntfHelpers.py:335
msgid "You are about to delete a RAID device."
msgstr "RAID eszköz törlésére készül."

#: partIntfHelpers.py:337
#, python-format
msgid "You are about to delete the %s partition."
msgstr "%s rész törlésére készül."

#: partIntfHelpers.py:341
#, python-format
msgid "You are about to delete the %(type)s %(name)s"
msgstr ""

#: partIntfHelpers.py:352
msgid "Confirm Reset"
msgstr "Visszaállítás megerősítése"

#: partIntfHelpers.py:353
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr "Biztosan vissza akarja állítani a partíciós tábla eredeti állapotát?"

#: partIntfHelpers.py:362
msgid "No authentication"
msgstr ""

#: partIntfHelpers.py:363
msgid "CHAP pair"
msgstr ""

#: partIntfHelpers.py:364
msgid "CHAP pair and a reverse pair"
msgstr ""

#: partIntfHelpers.py:365
msgid "Use the credentials from the discovery step"
msgstr ""

#: partIntfHelpers.py:472
msgid "iSCSI Nodes"
msgstr ""

#: partIntfHelpers.py:473
msgid "No iSCSI nodes to log in"
msgstr ""

#: partIntfHelpers.py:516
msgid "iSCSI Error"
msgstr ""

#: partIntfHelpers.py:521
msgid "Discovery"
msgstr ""

#: partIntfHelpers.py:521
msgid "Login"
msgstr ""

#: partIntfHelpers.py:522
#, python-format
msgid "iSCSI %s Error"
msgstr ""

#: platform.py:100 platform.py:403 platform.py:471 platform.py:592
#: ui/create-storage.glade.h:15
msgid "RAID Device"
msgstr "RAID eszköz"

#: platform.py:101 platform.py:104 platform.py:404 platform.py:472
#: platform.py:593 platform.py:596
msgid "Master Boot Record (MBR)"
msgstr "Master Boot Record (MBR)"

#: platform.py:103 platform.py:595
msgid "First sector of boot partition"
msgstr "A rendszerindító rész első szektora"

#: platform.py:115 platform.py:278
msgid "You have not created a bootable partition."
msgstr "Nem csinált rendszerindító részt."

#: platform.py:119
msgid "Bootable partitions cannot be on a logical volume."
msgstr "Rendszerindító partíció nem lehet logikai köteten."

#: platform.py:124
msgid "Bootable partitions cannot be on a RAID device."
msgstr "Rendszerindító partíció nem lehet RAID eszközön."

#: platform.py:126
msgid "Bootable partitions can only be on RAID1 devices."
msgstr "Rendszerindító partíció csak RAID1 eszközön lehet."

#: platform.py:133
#, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr "Rendszerindító partíció nem lehet %s fájlrendszeren."

#: platform.py:137 platform.py:142
msgid "Bootable partitions cannot be on an encrypted block device"
msgstr "Rendszerindító partíció nem lehet titkosított eszközön."

#: platform.py:264
msgid "EFI System Partition"
msgstr "EFI rendszer partíció"

#: platform.py:276
msgid "You have not created a /boot/efi partition."
msgstr "Nem csinált /boot/efi részt."

#: platform.py:284
msgid "/boot/efi is not EFI."
msgstr "/boot/efi nem EFI."

#: platform.py:306
#, python-format
msgid "%s must have a %s disk label."
msgstr ""

#: platform.py:338
#, python-format
msgid "%s must have a bsd disk label."
msgstr "%s résznek rendelkeznie kell BSD lemez címkével."

#: platform.py:350
#, python-format
msgid "The disk %s requires at least 1MB of free space at the beginning."
msgstr "%s lemez elején legalább 1MB szabad helynek kell lennie."

#: platform.py:376
msgid "The boot partition must be a primary partition."
msgstr ""

#: platform.py:406
msgid "PPC PReP Boot"
msgstr "PPC PReP rendszerindító"

#: platform.py:422
msgid "The boot partition must be within the first 4MB of the disk."
msgstr "A boot résznek a lemez első 4MB-jában kell lennie."

#: platform.py:474 platform.py:477
msgid "Apple Bootstrap"
msgstr "Apple rendszerindító"

#: platform.py:491
#, python-format
msgid "%s must have a mac disk label."
msgstr "%s résznek kell egy mac lemez címke."

#: rescue.py:221
msgid "When finished please exit from the shell and your system will reboot."
msgstr ""
"Amint elkészült, kérem lépjen ki a rendszerhéjból, és a rendszer újra fog "
"indulni."

#: rescue.py:235
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr "Nem találom a futtatandó /bin/sh-t!  Nem indítok parancssort"

#: rescue.py:254
msgid "Setup Networking"
msgstr "Hálózat beállítása"

#: rescue.py:255
msgid "Do you want to start the network interfaces on this system?"
msgstr "El szeretné indítani a rendszer hálózati csatolóit?"

#: rescue.py:256 loader/driverdisk.c:740 loader/driverdisk.c:750
#: loader/hdinstall.c:214 textw/constants_text.py:56
msgid "Yes"
msgstr "Igen"

#: rescue.py:256 rescue.py:258 loader/driverdisk.c:740 loader/driverdisk.c:750
#: textw/constants_text.py:60
msgid "No"
msgstr "Nem"

#: rescue.py:263
msgid ""
"Unable to activate a networking device.  Networking will not be available in "
"rescue mode."
msgstr ""
"Nem lehet a hálózati eszközt bekapcsolni.  Hálózat nem lesz elérhető mentő "
"módban."

#: rescue.py:299 rescue.py:374 rescue.py:385 rescue.py:465 rescue.py:494
msgid "Rescue"
msgstr "Mentés"

#: rescue.py:300
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.  If you need to activate SAN devices choose "
"'Advanced'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""

#: rescue.py:311 iw/partition_gui.py:787 loader/cdinstall.c:218
#: loader/cdinstall.c:226 loader/driverdisk.c:695 storage/__init__.py:165
#: storage/devicetree.py:93
msgid "Continue"
msgstr "Folytatás"

#: rescue.py:311 rescue.py:320
msgid "Read-Only"
msgstr "Csak olvasható"

#: rescue.py:311 rescue.py:313 loader/cdinstall.c:260 loader/cdinstall.c:263
#: loader/method.c:324 storage/__init__.py:1905 textw/upgrade_text.py:142
msgid "Skip"
msgstr "Kihagyás"

#: rescue.py:311 rescue.py:316
msgid "Advanced"
msgstr ""

#: rescue.py:350
msgid "System to Rescue"
msgstr "Mentendő rendszer"

#: rescue.py:351
msgid "Which device holds the root partition of your installation?"
msgstr "Melyik rész tartalmazza a meglévő rendszer gyökerét?"

#: rescue.py:353 rescue.py:357 text.py:633 text.py:635
msgid "Exit"
msgstr "Kilépés"

#: rescue.py:375
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions.  "
"The system will reboot automatically when you exit from the shell."
msgstr ""
"A rendszerben rendezetlen fájlrendszerek vannak, amiket úgy döntött, nem "
"csatlakoztat.  Nyomjon Enter-t, hogy a parancssorban a részeket fsck-val "
"kijavíthatja, és becsatolhatja. A rendszer magától újraindul, amikor kilép a "
"rendszerhéjból."

#: rescue.py:386
#, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"The system will reboot automatically when you exit from the shell."
msgstr ""

#: rescue.py:466
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell. The system will reboot automatically when you "
"exit from the shell."
msgstr ""
"A rendszer néhány része vagy egésze becsatolása alatt hiba történt. Néhány "
"még becsatolva maradhat %s alatt.\n"
"\n"
"Rendszerhéj megnyitásához nyomjon <Enter>-t. A rendszerhéjból kilépve a gép "
"magától újra fog indulni."

#: rescue.py:476
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr "Nincs semmilyen Linux partíciója.  Újraindulás.\n"

#: rescue.py:479
msgid "Rescue Mode"
msgstr "Mentő mód"

#: rescue.py:480
msgid ""
"You don't have any Linux partitions. Press return to get a shell. The system "
"will reboot automatically when you exit from the shell."
msgstr ""
"Nincs Linux partíció. Nyomjon Entert a parancssorhoz. A rendszer magától "
"újraindul, amikor kilép a parancssorból."

#: rescue.py:493
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "A rendszer becsatolva %s ponton."

#: text.py:154
msgid "Passphrase for encrypted device"
msgstr "Jelszó a titkosított részhez"

#: text.py:167
msgid "Also add this passphrase to all existing encrypted devices"
msgstr "E jelszó használata minden meglévő titkosított eszközhöz"

#: text.py:198
#, python-format
msgid "The passphrase must be at least %d character long."
msgid_plural "The passphrase must be at least %d characters long."
msgstr[0] ""
msgstr[1] ""

#: text.py:232 ui/lukspassphrase.glade.h:6
msgid "Passphrase"
msgstr "Jelszó"

#: text.py:240 ui/lukspassphrase.glade.h:7
msgid "This is a global passphrase"
msgstr "Ez globális jelszó"

#: text.py:360 text.py:364
msgid "Repository editing is not available in text mode."
msgstr "Tároló szerkesztés nem érhető el szöveges módban."

#: text.py:427
#, python-format
msgid "Welcome to %(productName)s for %(productArch)s"
msgstr ""

#: text.py:429
#, python-format
msgid "Welcome to %s"
msgstr "Üdvözli %s"

#: text.py:431
msgid ""
"  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next "
"screen"
msgstr ""
"<Tab>/<Alt-Tab> elemek közti váltás  |  <Szóköz>  kiválasztás  |  <F12> köv. "
"képernyő"

#: text.py:491
#, python-format
msgid ""
"Error processing drive:\n"
"\n"
"%(path)s\n"
"%(size)-0.fMB\n"
"%(description)s\n"
"\n"
"This device may need to be reinitialized.\n"
"\n"
"REINITIALIZING WILL CAUSE ALL DATA TO BE LOST!\n"
"\n"
"This action may also be applied to all other disks needing reinitialization.%"
"(details)s"
msgstr ""

#: text.py:633 loader/net.c:112 loader/net.c:554 loader/net.c:696
#: loader/net.c:704 loader/net.c:1130 loader/net.c:1138
#: textw/constants_text.py:68
msgid "Retry"
msgstr "Újra"

#: text.py:659
msgid "Cancelled"
msgstr "Törölve"

#: text.py:660
msgid "I can't go to the previous step from here. You will have to try again."
msgstr "Innen nem lehet visszalépni. Újra meg kell próbálnia."

#: upgrade.py:51
msgid "Proceed with upgrade?"
msgstr "Frissítéssel folytatja?"

#: upgrade.py:52
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have "
"already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""
"A frissíteni kívánt Linux telepítés fájlrendszereit már csatlakoztattam. "
"Ettől a ponttól már nem léphet vissza. \n"
"\n"

#: upgrade.py:56
msgid "Would you like to continue with the upgrade?"
msgstr "Folytatni szeretné a rendszer frissítését?"

#: upgrade.py:177
msgid "Mount failed"
msgstr "Becsatolás sikertelen"

#: upgrade.py:178
#, python-format
msgid ""
"The following error occurred when mounting the file systems listed in /etc/"
"fstab.  Please fix this problem and try to upgrade again.\n"
"%s"
msgstr ""
"Az /etc/fstab fájlban felsorolt részeket becsatolva a következő hiba "
"történt.  Kérem javítsa ki ezt a hibát, majd próbálkozzon a frissítéssel "
"újra.\n"
"%s"

#: upgrade.py:186 upgrade.py:193
msgid "Upgrade root not found"
msgstr "Frissíthető gyökér nem található"

#: upgrade.py:187
msgid "The root for the previously installed system was not found."
msgstr "A korábban telepített rendszer root-ja nem található."

#: upgrade.py:194
msgid ""
"The root for the previously installed system was not found.  You can exit "
"installer or backtrack to choose installation instead of upgrade."
msgstr ""
"Az előző rendszer gyökerét nem találtam.  Kiléphet a telepítőből, vagy "
"visszaléphet, hogy a telepítést válassza a frissítés helyett."

#: upgrade.py:217
msgid ""
"The following files are absolute symbolic links, which we do not support "
"during an upgrade. Please change them to relative symbolic links and restart "
"the upgrade.\n"
"\n"
msgstr ""
"A következő fájlok abszolút szimbolikus linkek, melyeket a frissítési "
"folyamat nem támogat. Kérem változtassa meg ezeket relatív szimbolikus "
"linkekre, és indítsa újra a frissítési folyamatot.\n"
"\n"

#: upgrade.py:223
msgid "Absolute Symlinks"
msgstr "Abszolút szimbolikus linkek"

#: upgrade.py:234
msgid ""
"The following are directories which should instead be symbolic links, which "
"will cause problems with the upgrade.  Please return them to their original "
"state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""
"A következő mappák a várttal ellentétben nem szimbolikus linkek, ami gondot "
"okoz frissítés  közben.  Kérem állítsa vissza az eredeti állapotukat "
"szimbolikus linkekként, majd indítsa újra a frissítést.\n"
"\n"

#: upgrade.py:240
msgid "Invalid Directories"
msgstr "Érvénytelen mappák"

#: vnc.py:136
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr ""

#: vnc.py:142
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr ""

#: vnc.py:161
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "Csatlakozás %s gazdán levő vnc ügyfélhez..."

#: vnc.py:175
msgid "Connected!"
msgstr "Kapcsolódva!"

#: vnc.py:178
msgid "Will try to connect again in 15 seconds..."
msgstr "Újabb próbálkozás 15 másodperc múlva..."

#: vnc.py:184
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] ""
msgstr[1] ""

#: vnc.py:195
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""
"Kérem csatlakozzon a vnc ügyféllel %s gazdához, hogy megkezdje a telepítést."

#: vnc.py:197
msgid "Please manually connect your vnc client to begin the install."
msgstr "Kérem csatlakozzon vnc ügyféllel, hogy megkezdje a telepítést."

#: vnc.py:200
msgid "Starting VNC..."
msgstr "VNC indítása..."

#: vnc.py:237
msgid "The VNC server is now running."
msgstr "A VNC kiszolgáló elindult."

#: vnc.py:241
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Ön egy figyelő vncviewerhez csatlakozást választja. \n"
"Ez nem igényel beállított jelszót.  Ha beállít egy jelszót, \n"
"akkor azt használni fogom, amennyiben a csatlakozás \n"
"a vncviewerhez sikertelen\n"
"\n"

#: vnc.py:246
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"FIGYELEM!!! A VNC kiszolgáló JELSZÓ NÉLKÜL működik!\n"
"Használhatja a vncpassword=<jelszó> rendszerindító \n"
"beállítást, ha biztonságossá szeretné tenni a kiszolgálót.\n"
"\n"

#: vnc.py:250
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"Ön vnc futtatását választja jelszóval. \n"
"\n"

#: vnc.py:252
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"Ismeretlen hiba.  Kilépés. \n"
"\n"

#: vnc.py:274 vnc.py:367
msgid "VNC Configuration"
msgstr "VNC beállítása"

#: vnc.py:277 vnc.py:371
msgid "No password"
msgstr "Nincs jelszó"

#: vnc.py:279 vnc.py:374
msgid ""
"A password will prevent unauthorized listeners connecting and monitoring "
"your installation progress.  Please enter a password to be used for the "
"installation"
msgstr ""
"Jelszó használatával lehet megakadályozni, hogy illetéktelenek férjenek "
"hozzá a telepítési folyamathoz.  Kérem adjon meg a telepítéshez használandó "
"jelszót"

#: vnc.py:287 vnc.py:382 textw/userauth_text.py:47
msgid "Password:"
msgstr "Jelszó:"

#: vnc.py:288 vnc.py:383 textw/userauth_text.py:49
msgid "Password (confirm):"
msgstr "Jelszó (megerősítés):"

#: vnc.py:306 vnc.py:405 textw/userauth_text.py:70
msgid "Password Mismatch"
msgstr "Eltérő jelszavak"

#: vnc.py:307 vnc.py:406 textw/userauth_text.py:71
msgid "The passwords you entered were different. Please try again."
msgstr "Az ön által beírt jelszók különböznek. Kérem próbálja újra."

#: vnc.py:312 vnc.py:411 textw/userauth_text.py:66
msgid "Password Length"
msgstr "Jelszó hossz"

#: vnc.py:313 vnc.py:412
msgid "The password must be at least six characters long."
msgstr "A jelszó legalább 6 karakter hosszú kell legyen."

#: vnc.py:335
msgid "Unable to Start X"
msgstr "Nem lehet az X-et elindítani"

#: vnc.py:337
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""
"Az X nem tudott elindulni az Ön gépén.  El szeretné indítani a VNC-t, hogy "
"egy másik gépről csatlakozni lehessen ehhez a géphez - a grafikus módú "
"telepítés indításához, vagy inkább szöveges módban szeretné folytatni a "
"telepítést?"

#: vnc.py:356
msgid "Start VNC"
msgstr "VNC indítása"

#: vnc.py:357 vnc.py:359
msgid "Use text mode"
msgstr "Szöveges mód használata"

#: yuminstall.py:87
#, python-format
msgid "%s MB"
msgstr "%s MB"

#: yuminstall.py:90
#, python-format
msgid "%s KB"
msgstr "%s kB"

#: yuminstall.py:92
#, python-format
msgid "%s Byte"
msgid_plural "%s Bytes"
msgstr[0] ""
msgstr[1] ""

#: yuminstall.py:136
msgid "Preparing to install"
msgstr "Telepítés előkészítése"

#: yuminstall.py:137
msgid "Preparing transaction from installation source"
msgstr "Tranzakció előkészítése telepítő forrásból"

#: yuminstall.py:165
#, python-format
msgid "<b>Installing %(pkgStr)s</b> (%(size)s)\n"
msgstr ""

#: yuminstall.py:230
#, python-format
msgid "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgid_plural "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgstr[0] ""
msgstr[1] ""

#: yuminstall.py:248
msgid "Finishing upgrade"
msgstr "Felfejlesztés befejezése"

#: yuminstall.py:249
msgid "Finishing upgrade process.  This may take a little while."
msgstr "Felfejlesztés befejezése.  Ez egy ideig eltarthat."

#: yuminstall.py:266
msgid "Error Installing Package"
msgstr "Csomag telepítése meghiúsult"

#: yuminstall.py:267
#, python-format
msgid ""
"A fatal error occurred when installing the %s package.  This could indicate "
"errors when reading the installation media.  Installation cannot continue."
msgstr ""
"Végzetes hiba történt a %s csomag telepítése közben. Ez jelentheti a "
"telepítő média hibáit is ami olvasás közben derült ki. Telepítés nem tud "
"folytatódni."

#: yuminstall.py:428
msgid "Change Disc"
msgstr "Cseréljen lemezt"

#: yuminstall.py:429
#, python-format
msgid "Please insert %(productName)s disc %(discnum)d to continue."
msgstr ""

#: yuminstall.py:439
msgid "Wrong Disc"
msgstr "Rossz lemez"

#: yuminstall.py:440
#, python-format
msgid "That's not the correct %s disc."
msgstr "Ez nem a megfelelő %s lemez."

#: yuminstall.py:447
msgid "Unable to access the disc."
msgstr "Elérhetetlen a lemez."

#: yuminstall.py:611
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr "%r tároló neve hiányzik a beállításban, id-t használók"

#: yuminstall.py:964 yuminstall.py:966
msgid "Re_boot"
msgstr "Újrain_dítás"

#: yuminstall.py:964
msgid "_Eject"
msgstr "L_emez kiadása"

#: yuminstall.py:983
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt "
"package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""
"%s fájlt nem sikerült megnyitni.  Valószínűleg hiányzó fájl, sérült csomag, "
"vagy sérült hordozó okozta.  Kérem ellenőrizze a telepítő forrást.\n"
"\n"
"Ha kilép, a rendszer töredékkel teli marad, s emiatt újbóli telepítést "
"igényel.\n"
"\n"

#: yuminstall.py:1031
msgid "Retrying"
msgstr "Újrapróbálás"

#: yuminstall.py:1031
msgid "Retrying download."
msgstr "Letöltés újrapróbálása."

#: yuminstall.py:1093
#, python-format
msgid ""
"There was an error running your transaction for the following reason: %s\n"
msgstr "Az átvitel végrehajtása alatt hiba történt a következő okból: %s\n"

#: yuminstall.py:1138 yuminstall.py:1139
msgid "file conflicts"
msgstr "fájl ütközés"

#: yuminstall.py:1140
msgid "older package(s)"
msgstr "régebbi csomag(ok)"

#: yuminstall.py:1141
msgid "insufficient disk space"
msgstr "nem elegendő szabad lemezterület"

#: yuminstall.py:1142
msgid "insufficient disk inodes"
msgstr "nincs elég inode"

#: yuminstall.py:1143
msgid "package conflicts"
msgstr "csomag ütközés"

#: yuminstall.py:1144
msgid "package already installed"
msgstr "már telepített csomag"

#: yuminstall.py:1145
msgid "required package"
msgstr "szükséges csomag"

#: yuminstall.py:1146
msgid "package for incorrect arch"
msgstr "nem megfelelő architektúrához készült csomag"

#: yuminstall.py:1147
msgid "package for incorrect os"
msgstr "nem megfelelő operációs rendszerhez készült csomag"

#: yuminstall.py:1161
msgid "You need more space on the following file systems:\n"
msgstr "Több szabad helyre van szüksége az alábbi fájlrendszereken:\n"

#: yuminstall.py:1174
#, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr ""
"Fájlütközés történt a telepítendő csomagokat ellenőrzése alatt:\n"
"%s\n"

#: yuminstall.py:1177
#, python-format
msgid ""
"There was an error running your transaction for the following reason(s): %"
"s.\n"
msgstr "Az átvitel végrehajtása alatt hiba történt a következő miatt: %s\n"

#: yuminstall.py:1185 yuminstall.py:1190
msgid "Error Running Transaction"
msgstr "Hiba az átvitel végrehajtása alatt"

#: yuminstall.py:1341 iw/task_gui.py:312
msgid ""
"Some of your software repositories require networking, but there was an "
"error enabling the network on your system."
msgstr ""
"Néhány szoftver tárolóhoz hálózat kell, de a hálózat engedélyezése közben "
"hiba történt ."

#: yuminstall.py:1377
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""
"Nem sikerült beolvasni a csoportinformációt a tárolókból. A probléma a "
"telepítési fa felépítésében lépett fel."

#: yuminstall.py:1433 textw/constants_text.py:64
msgid "Edit"
msgstr "Szerkesztés"

#: yuminstall.py:1444 yuminstall.py:1642
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.\n"
"\n"
"%s"
msgstr ""
"Nem lehet beolvasni a csomagleírásokat. Ezt okozhatja a repodata mappa "
"hiánya.  Kérem ellenőrizze, hogy jól készítették-e el a telepítő fát.\n"
"\n"
"%s"

#: yuminstall.py:1622
msgid ""
"Some of the packages you have selected for install are missing "
"dependencies.  You can exit the installation, go back and change your "
"package selections, or continue installing these packages without their "
"dependencies."
msgstr ""

#: yuminstall.py:1664
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or exit the "
"installer."
msgstr ""
"A kiválasztott csomagoknak %d MB szabad hely kell, de nincs elég elérhető.  "
"Megváltoztathatja az összeállítást, vagy kiléphet a telepítőből."

#: yuminstall.py:1685
msgid "Reboot?"
msgstr "Újraindítás?"

#: yuminstall.py:1686
msgid "The system will be rebooted now."
msgstr "A rendszer most újraindul."

#: yuminstall.py:1831
#, python-format
msgid ""
"You appear to be upgrading from a system which is too old to upgrade to this "
"version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr ""
"Úgy tűnik, a rendszer túl régi, ezért nem frissíthető fel %s változatra.  "
"Biztosan folytatni szeretné a folyamatot?"

#: yuminstall.py:1866
#, python-format
msgid ""
"The arch of the release of %(productName)s you are upgrading to appears to "
"be %(myarch)s which does not match your previously installed arch of %(arch)"
"s.  This is likely to not succeed.  Are you sure you wish to continue the "
"upgrade process?"
msgstr ""

#: yuminstall.py:1913
msgid "Post Upgrade"
msgstr "Zárófrissítés"

#: yuminstall.py:1914
msgid "Performing post-upgrade configuration"
msgstr "Zárófrissítés beállításainak végrehajtása"

#: yuminstall.py:1916
msgid "Post Installation"
msgstr "Zárótelepítés"

#: yuminstall.py:1917
msgid "Performing post-installation configuration"
msgstr "Zárótelepítés beállításainak végrehajtása"

#: yuminstall.py:2163
msgid "Installation Starting"
msgstr "Kezdődik a telepítés"

#: yuminstall.py:2164
msgid "Starting installation process"
msgstr "Telepítési folyamat indul"

#: yuminstall.py:2202
msgid "Dependency Check"
msgstr "Függőség ellenőrzés"

#: yuminstall.py:2203
msgid "Checking dependencies in packages selected for installation"
msgstr "A telepítésre kiválasztott csomagok közötti függőségek ellenőrzése"

#: yuminstall.py:2259 yuminstall.py:2275
msgid "Retrieving installation information."
msgstr "Telepítő információk beszerzése."

#: yuminstall.py:2261 yuminstall.py:2277
#, python-format
msgid "Retrieving installation information for %s."
msgstr "Telepítő információk beszerzése %s számára."

#: yuminstall.py:2262 yuminstall.py:2278
msgid "Installation Progress"
msgstr "Telepítés állapota"

#: installclasses/fedora.py:38
msgid "_Fedora"
msgstr "_Fedora"

#: installclasses/fedora.py:39
#, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage. You can optionally select a different set of "
"software now."
msgstr ""

#: installclasses/fedora.py:48
msgid "Graphical Desktop"
msgstr ""

#: installclasses/fedora.py:53
msgid "Software Development"
msgstr "Szoftver fejlesztés"

#: installclasses/fedora.py:59
msgid "Web Server"
msgstr "Web kiszolgáló"

#: installclasses/fedora.py:63 installclasses/rhel.py:44
msgid "Minimal"
msgstr ""

#: installclasses/rhel.py:33
msgid "Red Hat Enterprise Linux"
msgstr "Red Hat Enterprise Linux"

#: installclasses/rhel.py:34
#, python-format
msgid ""
"The default installation of %s is a minimum install. You can optionally "
"select a different set of software now."
msgstr ""

#: iw/GroupSelector.py:147
#, python-format
msgid "Packages in %s"
msgstr "%s csomagok"

#: iw/GroupSelector.py:433
#, python-format
msgid "Optional packages selected: %(inst)d of %(cnt)d"
msgstr ""

#: iw/GroupSelector.py:435
#, python-format
msgid "<i>%s</i>"
msgstr "<i>%s</i>"

#: iw/GroupSelector.py:495
msgid "Uncategorized"
msgstr "Nem besorolt"

#: iw/account_gui.py:52
msgid "Root _Password:"
msgstr "_Rendszergazda jelszó:"

#: iw/account_gui.py:54
msgid "_Confirm:"
msgstr "_Megerősítés:"

#: iw/account_gui.py:92
msgid "Caps Lock is on."
msgstr "Caps Lock bekapcsolva."

#: iw/account_gui.py:102 iw/account_gui.py:110 iw/account_gui.py:117
#: iw/account_gui.py:138 textw/userauth_text.py:74
msgid "Error with Password"
msgstr "Jelszóhiba"

#: iw/account_gui.py:103
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""
"Meg kell adni a rendszergazda jelszót, majd megerősíteni ellenőrzésképpen, "
"mielőtt folytatja."

#: iw/account_gui.py:111
msgid "The passwords you entered were different.  Please try again."
msgstr "A megadott jelszók különböznek. Kérem próbálja újra."

#: iw/account_gui.py:118
msgid "The root password must be at least six characters long."
msgstr "Rendszergazda jelszó legalább 6 karakter hosszú kell legyen."

#: iw/account_gui.py:127 textw/userauth_text.py:83
msgid "Weak Password"
msgstr "Gyenge jelszó"

#: iw/account_gui.py:128 textw/userauth_text.py:84
#, python-format
msgid "You have provided a weak password: %s"
msgstr "Gyenge jelszót adott meg: %s"

#: iw/account_gui.py:131 textw/userauth_text.py:87
msgid "_Use Anyway"
msgstr ""

#: iw/account_gui.py:139 textw/userauth_text.py:75
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr "A kért jelszó tartalmaz nem ASCII jeleket, ami nem engedélyezett."

#: iw/advanced_storage.py:260
msgid "Node Name"
msgstr ""

#: iw/advanced_storage.py:261
msgid "Interface"
msgstr ""

#: iw/advanced_storage.py:386
msgid "You must select a NIC to use."
msgstr ""

#: iw/autopart_type.py:96
msgid ""
"No partitions are available to resize.  Only physical partitions with "
"specific filesystems can be resized."
msgstr ""
"Nincs átméretezésre elérhető lemezrész.  Csak bizonyos fájlrendszerű "
"részeket lehet átméretezni."

#: iw/autopart_type.py:121
msgid "Resize FileSystem Error"
msgstr "Fájlrendszer átméretezés hiba"

#: iw/autopart_type.py:122
#, python-format
msgid "%(device)s: %(msg)s"
msgstr ""

#: iw/autopart_type.py:131
msgid "Resize Device Error"
msgstr "Eszköz átméretezés hiba"

#: iw/autopart_type.py:132
#, python-format
msgid "%(name)s: %(msg)s"
msgstr ""

#: iw/autopart_type.py:257
msgid "Use All Space"
msgstr ""

#: iw/autopart_type.py:259
msgid ""
"Removes all partitions on the selected device(s).  This includes partitions "
"created by other operating systems.\n"
"\n"
"<b>Tip:</b> This option will remove data from the selected device(s).  Make "
"sure you have backups."
msgstr ""

#: iw/autopart_type.py:265
msgid "Replace Existing Linux System(s)"
msgstr ""

#: iw/autopart_type.py:267
msgid ""
"Removes only Linux partitions (created from a previous Linux installation).  "
"This does not remove other partitions you may have on your storage device(s) "
"(such as VFAT or FAT32).\n"
"\n"
"<b>Tip:</b> This option will remove data from the selected device(s).  Make "
"sure you have backups."
msgstr ""

#: iw/autopart_type.py:275
msgid "Shrink Current System"
msgstr ""

#: iw/autopart_type.py:277
msgid ""
"Shrinks existing partitions to create free space for the default layout."
msgstr ""

#: iw/autopart_type.py:279
msgid "Use Free Space"
msgstr ""

#: iw/autopart_type.py:281
msgid ""
"Retains your current data and partitions and uses only the unpartitioned "
"space on the selected device(s), assuming you have enough free space "
"available."
msgstr ""

#: iw/autopart_type.py:285
msgid "Create Custom Layout"
msgstr ""

#: iw/autopart_type.py:287
msgid ""
"Manually create your own custom layout on the selected device(s) using our "
"partitioning tool."
msgstr ""

#: iw/blpasswidget.py:44
msgid "_Use a boot loader password"
msgstr "Jelszó h_asználata a rendszerindítóban"

#: iw/blpasswidget.py:45
msgid ""
"A boot loader password prevents users from changing kernel options, "
"increasing security."
msgstr ""
"A rendszerindító jelszavas védelme a biztonságot növelő lehetőség - "
"megakadályozza a felhasználókat, hogy a rendszermag paramétereit elállítsák."

#: iw/blpasswidget.py:76
msgid "Change _password"
msgstr "_Jelszó megváltoztatása"

#: iw/blpasswidget.py:99
msgid "Enter Boot Loader Password"
msgstr "Adja meg a rendszerindító jelszót"

#: iw/blpasswidget.py:105
msgid ""
"Enter a boot loader password and then confirm it.  (Note that your BIOS "
"keymap may be different than the actual keymap you are used to.)"
msgstr ""
"Adja meg a rendszerindító jelszót, és erősítse meg.  (Ne felejtse el, hogy a "
"BIOS esetleg más billentyűzetkiosztást használ, mint amit ön megszokott.)"

#: iw/blpasswidget.py:112
msgid "_Password:"
msgstr "J_elszó:"

#: iw/blpasswidget.py:118
msgid "Con_firm:"
msgstr "_Megerősítés:"

#: iw/blpasswidget.py:139
msgid "Passwords don't match"
msgstr "A jelszavak különböznek"

#: iw/blpasswidget.py:140
msgid "Passwords do not match"
msgstr "A jelszavak nem egyeznek meg"

#: iw/blpasswidget.py:147
msgid "Password empty"
msgstr ""

#: iw/blpasswidget.py:148
msgid "You did not enter a password"
msgstr ""

#: iw/blpasswidget.py:155
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a "
"longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""
"A rendszerindító jelszó rövidebb 6 karakternél.  Ennél hosszabb jelszót "
"célszerű választani.\n"
"\n"
"Folytatni szeretné ezzel a jelszóval?"

#: iw/bootloader_main_gui.py:37
msgid "Boot Loader Configuration"
msgstr "Rendszerindító beállítás"

#: iw/bootloader_main_gui.py:182 iw/bootloader_main_gui.py:187
#: iw/bootloader_main_gui.py:225
#, python-format
msgid "_Install boot loader on /dev/%s."
msgstr "_Rendszerindító telepítése a /dev/%s eszközre."

#: iw/bootloader_main_gui.py:235
msgid "_Change device"
msgstr "Eszköz_csere"

#: iw/cleardisks_gui.py:34
msgid "Clear Disks Selector"
msgstr ""

#: iw/cleardisks_gui.py:45 iw/filter_gui.py:438
msgid "You must select at least one drive to be used for installation."
msgstr ""

#: iw/cleardisks_gui.py:55
msgid "You must select one drive to boot from."
msgstr ""

#: iw/cleardisks_gui.py:127 iw/cleardisks_gui.py:147 iw/filter_gui.py:504
#: iw/filter_gui.py:516 iw/filter_gui.py:546
msgid "Model"
msgstr ""

#: iw/cleardisks_gui.py:128 iw/cleardisks_gui.py:148
msgid "Capacity"
msgstr ""

#: iw/cleardisks_gui.py:129 iw/filter_gui.py:506 iw/filter_gui.py:526
#: iw/filter_gui.py:537 iw/filter_gui.py:548
msgid "Vendor"
msgstr ""

#: iw/cleardisks_gui.py:130 iw/cleardisks_gui.py:149 iw/filter_gui.py:509
#: iw/filter_gui.py:524 iw/filter_gui.py:535 iw/filter_gui.py:551
msgid "Identifier"
msgstr ""

#: iw/cleardisks_gui.py:131 iw/filter_gui.py:507 iw/filter_gui.py:527
#: iw/filter_gui.py:538 iw/filter_gui.py:549
msgid "Interconnect"
msgstr ""

#: iw/cleardisks_gui.py:145
msgid ""
"Boot\n"
"Loader"
msgstr ""

#: iw/cleardisks_gui.py:195
msgid ""
"<b>Tip:</b> All Linux filesystems on install target devices will be "
"reformatted and wiped of any data.  Make sure you have backups."
msgstr ""

#: iw/cleardisks_gui.py:197 ui/cleardisks.glade.h:3
msgid ""
"<b>Tip:</b> Install target devices will be reformatted and wiped of any "
"data.  Make sure you have backups."
msgstr ""

#: iw/cleardisks_gui.py:199
msgid ""
"<b>Tip:</b> Your filesystems on install target devices will not be wiped "
"unless you choose to do so during customization."
msgstr ""

#: iw/congrats_gui.py:33
msgid "Congratulations"
msgstr "Gratulálok"

#: iw/congrats_gui.py:74 textw/complete_text.py:39
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
msgstr ""
"Gratulálok! %s telepítése kész.\n"
"\n"

#: iw/congrats_gui.py:77 textw/complete_text.py:42
msgid "Shutdown"
msgstr "Leállítás"

#: iw/congrats_gui.py:79 textw/complete_text.py:44
msgid "Please shutdown to use the installed system.\n"
msgstr ""
"Kérem állítsa le a rendszert, hogy a telepített rendszert használhassa.\n"

#: iw/congrats_gui.py:81 textw/complete_text.py:46
msgid "Please reboot to use the installed system.\n"
msgstr ""
"Kérem indítsa újra a rendszert, hogy a telepített rendszert használhassa.\n"

#: iw/congrats_gui.py:86 textw/complete_text.py:51
msgid ""
"Note that updates may be available to ensure the proper functioning of your "
"system and installation of these updates is recommended after the reboot."
msgstr ""
"Vegye észre, hogy a rendszer hibátlan működésének biztosításához frissítések "
"lehetnek elérhetők, amiket ajánlunk az újraindulás után."

#: iw/congrats_gui.py:90 textw/complete_text.py:55
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"Please reboot to use the installed system.  Note that updates may be "
"available to ensure the proper functioning of your system and installation "
"of these updates is recommended after the reboot."
msgstr ""
"Végeztünk %s telepítésével. Gratulálok!\n"
"\n"
"Kérem indítson újra, hogy a telepített rendszert használja.  Vegye észre, "
"hogy frissítések lehetnek elérhetők a rendszer megfelelő működéséhez, és "
"ezek telepítése ajánlott az újraindulás után."

#: iw/examine_gui.py:37
msgid "Upgrade Examine"
msgstr "Frissítési vizsgálat"

#: iw/examine_gui.py:58
msgid "Fresh Installation"
msgstr ""

#: iw/examine_gui.py:60
#, python-format
msgid ""
"Choose this option to install a fresh copy of %s on your system.  Existing "
"software and data may be overwritten depending on your configuration choices."
msgstr ""

#: iw/examine_gui.py:65
msgid "Upgrade an Existing Installation"
msgstr ""

#: iw/examine_gui.py:67
#, python-format
msgid ""
"Choose this option if you would like to upgrade your existing %s system.  "
"This option will preserve the existing data on your storage device(s)."
msgstr ""

#: iw/examine_gui.py:112
msgid ""
"At least one existing installation has been detected on your system.  What "
"would you like to do?"
msgstr ""

#: iw/examine_gui.py:131
#, python-format
msgid "<b>Which %s installation would you like to upgrade?</b>"
msgstr ""

#: iw/examine_gui.py:145
msgid "Unknown Linux system"
msgstr "Ismeretlen Linux rendszer"

#: iw/filter_gui.py:154
#, python-format
msgid "<b>%s device(s) (%s MB) selected</b> out of %s device(s) (%s MB) total."
msgstr ""

#: iw/filter_gui.py:404
msgid "Device Filter"
msgstr ""

#: iw/filter_gui.py:505 iw/filter_gui.py:517 iw/filter_gui.py:525
#: iw/filter_gui.py:536 iw/filter_gui.py:547
msgid "Capacity (MB)"
msgstr ""

#: iw/filter_gui.py:508 iw/filter_gui.py:539 iw/filter_gui.py:550
msgid "Serial Number"
msgstr ""

#: iw/filter_gui.py:510 iw/filter_gui.py:518 iw/filter_gui.py:529
#: iw/filter_gui.py:540 iw/filter_gui.py:555 iw/osbootwidget.py:66
#: iw/partition_gui.py:613
msgid "Device"
msgstr "Eszköz"

#: iw/filter_gui.py:528
msgid "Paths"
msgstr ""

#: iw/filter_gui.py:552
msgid "Port"
msgstr ""

#: iw/filter_gui.py:553
msgid "Target"
msgstr ""

#: iw/filter_gui.py:554
msgid "LUN"
msgstr ""

#: iw/filter_type.py:55
msgid "What type of devices will your installation involve?"
msgstr ""

#: iw/filter_type.py:61
msgid "Basic Storage Devices"
msgstr ""

#: iw/filter_type.py:62
msgid ""
"Installs or upgrades to typical types of storage devices.  If you're not "
"sure which option is right for you, this is probably it."
msgstr ""

#: iw/filter_type.py:66
msgid "Specialized Storage Devices"
msgstr ""

#: iw/filter_type.py:67
msgid ""
"Installs or upgrades to enterprise devices such as Storage Area Networks "
"(SANs). This option will allow you to add FCoE / iSCSI / zFCP disks and to "
"filter out devices the installer should ignore."
msgstr ""

#: iw/language_gui.py:33 textw/language_text.py:46
msgid "Language Selection"
msgstr "Nyelv kiválasztása"

#: iw/language_gui.py:81 loader/lang.c:371 textw/language_text.py:47
msgid "What language would you like to use during the installation process?"
msgstr "Melyik nyelvet szeretné használni a telepítés alatt?"

#: iw/lvm_dialog_gui.py:128 iw/lvm_dialog_gui.py:174 iw/lvm_dialog_gui.py:188
#: iw/lvm_dialog_gui.py:227 iw/lvm_dialog_gui.py:304 iw/lvm_dialog_gui.py:703
#: iw/lvm_dialog_gui.py:724
msgid "Not enough space"
msgstr "Nincs elég szabad hely"

#: iw/lvm_dialog_gui.py:129
msgid ""
"The physical extent size cannot be changed because otherwise the space "
"required by the currently defined logical volumes will be increased to more "
"than the available space."
msgstr ""
"A fizikai terjedelem mérete nem változtatható meg, mert akkor a már meglévő "
"logikai kötetek mérete meghaladná a rendelkezésre álló helyet."

#: iw/lvm_dialog_gui.py:138
msgid "Confirm Physical Extent Change"
msgstr "Fizikai terjedelem méretének változtatás megerősítése"

#: iw/lvm_dialog_gui.py:139
msgid ""
"This change in the value of the physical extent will require the sizes of "
"the current logical volume requests to be rounded up in size to an integer "
"multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""
"Fizikai terjedelem értékének megváltozása megköveteli, hogy a már meglevő "
"logikai kötet kérelmek mérete a fizikai kiterjedés méretének egész számú "
"többszörösére legyen felkerekítve.\n"
"\n"
"A változás azonnal érvénybe lép."

#: iw/lvm_dialog_gui.py:148 iw/lvm_dialog_gui.py:210
msgid "C_ontinue"
msgstr "F_olytatás"

#: iw/lvm_dialog_gui.py:175
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"(curpe)10.2f MB) is larger than the smallest physical volume (%(maxpvsize)"
"10.2f MB) in the volume group."
msgstr ""

#: iw/lvm_dialog_gui.py:189
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"(curpe)10.2f MB) is too large compared to the size of the smallest physical "
"volume (%(maxpvsize)10.2f MB) in the volume group."
msgstr ""

#: iw/lvm_dialog_gui.py:203
msgid "Too small"
msgstr "Túl kicsi"

#: iw/lvm_dialog_gui.py:204
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""
"Ha erre az értékre változtatja a fizikai kiterjedés méretét, akkor a kötet "
"csoporton belül legalább egy fizikai köteten jelentős helyet pocsékol el."

#: iw/lvm_dialog_gui.py:228
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of the currently "
"defined logical volumes."
msgstr ""
"Fizikai kiterjedés mérete nem változtatható meg, mert ezzel a maximális "
"logikai kötet méret (%10.2f MB) kisebb lenne, mint az egyik létező logikai "
"kötet mérete."

#: iw/lvm_dialog_gui.py:305
msgid ""
"You cannot remove this physical volume because otherwise the volume group "
"will be too small to hold the currently defined logical volumes."
msgstr ""
"E fizikai kötet nem távolítható el, mert akkor a logikai kötetcsoport túl "
"kicsi lenne ahhoz, hogy elférjenek rajta a most megadott logikai kötetek."

#: iw/lvm_dialog_gui.py:405
msgid "Make Logical Volume"
msgstr "Logikai kötet készítése"

#: iw/lvm_dialog_gui.py:407
#, python-format
msgid "Edit Logical Volume: %s"
msgstr "Logikai kötet szerkesztése: %s"

#: iw/lvm_dialog_gui.py:446 iw/raid_dialog_gui.py:449
msgid "_File System Type:"
msgstr "_Fájl rendszer típusa:"

#: iw/lvm_dialog_gui.py:452
msgid "_Logical Volume Name:"
msgstr "_Logikai kötet név:"

#: iw/lvm_dialog_gui.py:461 iw/partition_dialog_gui.py:502
msgid "_Size (MB):"
msgstr "Mé_ret (MB):"

#: iw/lvm_dialog_gui.py:468
#, python-format
msgid "(Max size is %s MB)"
msgstr "(Max. méret %s MB)"

#: iw/lvm_dialog_gui.py:473 iw/partition_dialog_gui.py:554
#: iw/partition_ui_helpers_gui.py:333 iw/raid_dialog_gui.py:421
msgid "_Encrypt"
msgstr "_Titkosítás"

#: iw/lvm_dialog_gui.py:482 iw/partition_dialog_gui.py:484
#: iw/raid_dialog_gui.py:459
msgid "Original File System Type:"
msgstr "Eredeti fájl rendszer típus:"

#: iw/lvm_dialog_gui.py:487 iw/partition_dialog_gui.py:492
#: iw/raid_dialog_gui.py:466
msgid "Original File System Label:"
msgstr "Eredeti fájl rendszer címke:"

#: iw/lvm_dialog_gui.py:492
msgid "Logical Volume Name:"
msgstr "Logikai kötet név:"

#: iw/lvm_dialog_gui.py:496
msgid "Size (MB):"
msgstr "Méret (MB):"

#: iw/lvm_dialog_gui.py:523 iw/partition_dialog_gui.py:435
#: iw/raid_dialog_gui.py:440
msgid "_Mount Point:"
msgstr "_Csatolási pont:"

#: iw/lvm_dialog_gui.py:604 iw/partition_dialog_gui.py:118
#: iw/raid_dialog_gui.py:171
msgid "Mount Point Error"
msgstr ""

#: iw/lvm_dialog_gui.py:615
msgid "Illegal Logical Volume Name"
msgstr "Tiltott logikai kötet név"

#: iw/lvm_dialog_gui.py:631
msgid "Illegal logical volume name"
msgstr "Tiltott logikai kötet név"

#: iw/lvm_dialog_gui.py:632
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr ""
"Ezt a „%s” logikai kötet nevet már használják. Kérem válasszon másikat."

#: iw/lvm_dialog_gui.py:674 iw/partition_dialog_gui.py:134
#: iw/raid_dialog_gui.py:187
msgid "Mount point in use"
msgstr "Használt csatoló pont"

#: iw/lvm_dialog_gui.py:675 iw/partition_dialog_gui.py:135
#: iw/raid_dialog_gui.py:188
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr "Ezt a „%s” csatoló pontot már használják. Kérem válasszon másikat."

#: iw/lvm_dialog_gui.py:690
msgid "Illegal size"
msgstr "Tiltott méret"

#: iw/lvm_dialog_gui.py:691
msgid "The requested size as entered is not a valid number greater than 0."
msgstr "A megadott méret érvényes pozitív szám kell lengyen."

#: iw/lvm_dialog_gui.py:704
#, python-format
msgid ""
"The current requested size (%(size)10.2f MB) is larger than the maximum "
"logical volume size (%(maxlv)10.2f MB). To increase this limit you can "
"create more Physical Volumes from unpartitioned disk space and add them to "
"this Volume Group."
msgstr ""

#: iw/lvm_dialog_gui.py:725
#, python-format
msgid ""
"The size entered for this logical volume (%(size)d MB) combined with the "
"size of the other logical volume(s) exceeds the size of the volume group (%"
"(tempvgsize)d MB). Please make the volume group larger or make the logical "
"volume smaller."
msgstr ""

#: iw/lvm_dialog_gui.py:839
msgid "No free slots"
msgstr "Nincs üres aljzat"

#: iw/lvm_dialog_gui.py:840
#, python-format
msgid "You cannot create more than %d logical volume per volume group."
msgid_plural "You cannot create more than %d logical volumes per volume group."
msgstr[0] ""
msgstr[1] ""

#: iw/lvm_dialog_gui.py:850
msgid "No free space"
msgstr "Nincs szabad hely"

#: iw/lvm_dialog_gui.py:851
msgid ""
"There is no room left in the volume group to create new logical volumes. To "
"add a logical volume you must reduce the size of one or more of the "
"currently existing logical volumes"
msgstr ""
"Nem maradt hely a kötetcsoportban új logikai kötet létrehozásához. Logikai "
"kötet hozzáadásához legalább az egyik meglévő logikai kötet méretét le kell "
"csökkenteni."

#: iw/lvm_dialog_gui.py:888
#, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr "Biztosan törölni szeretné „%s” logikai kötetet?"

#: iw/lvm_dialog_gui.py:1003
msgid "Invalid Volume Group Name"
msgstr "Érvénytelen kötet csoport név"

#: iw/lvm_dialog_gui.py:1013
msgid "Name in use"
msgstr "Név használatban"

#: iw/lvm_dialog_gui.py:1014
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr "„%s” kötet csoport nevet már használják. Kérem válasszon másikat."

#: iw/lvm_dialog_gui.py:1114 iw/partition_gui.py:1615
#: storage/partitioning.py:233 storage/partitioning.py:276
msgid "Error Partitioning"
msgstr "Hiba a particionáláskor"

#: iw/lvm_dialog_gui.py:1115
#, python-format
msgid ""
"Could not allocate requested logical volumes:\n"
"\n"
"%s."
msgstr ""

#: iw/lvm_dialog_gui.py:1321
msgid "Not enough physical volumes"
msgstr "Nincs elég fizikai kötet"

#: iw/lvm_dialog_gui.py:1322
msgid ""
"At least one unused physical volume partition is needed to create an LVM "
"Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then "
"select the \"LVM\" option again."
msgstr ""
"Legalább egy nem használt fizikai kötet rész szükséges LVM kötetcsoport "
"teremtéséhez.\n"
"\n"
"Teremtsen egy „fizikai kötet (LVM)” típusú részt vagy egy RAID csoportot "
"majd válassza újból az „LVM” opciót."

#: iw/lvm_dialog_gui.py:1333
msgid "Make LVM Volume Group"
msgstr "LVM kötet csoport készítése"

#: iw/lvm_dialog_gui.py:1336
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr "LVM kötet csoport szerkesztése: %s"

#: iw/lvm_dialog_gui.py:1338
msgid "Edit LVM Volume Group"
msgstr "LVM kötet csoport szerkesztése"

#: iw/lvm_dialog_gui.py:1354
msgid "_Volume Group Name:"
msgstr "Kötet cso_port név:"

#: iw/lvm_dialog_gui.py:1362
msgid "Volume Group Name:"
msgstr "Kötet csoport név:"

#: iw/lvm_dialog_gui.py:1370
msgid "_Physical Extent:"
msgstr "_Fizikai darab:"

#: iw/lvm_dialog_gui.py:1385
msgid "Physical Volumes to _Use:"
msgstr "_Használandó fizikai kötetek:"

#: iw/lvm_dialog_gui.py:1391
msgid "Used Space:"
msgstr "Felhasznált hely:"

#: iw/lvm_dialog_gui.py:1408
msgid "Reserved Space:"
msgstr ""

#: iw/lvm_dialog_gui.py:1428
msgid "Free Space:"
msgstr "Szabad hely:"

#: iw/lvm_dialog_gui.py:1446
msgid "Total Space:"
msgstr "Összes hely:"

#: iw/lvm_dialog_gui.py:1484
msgid "Logical Volume Name"
msgstr "Logikai kötet név"

#: iw/lvm_dialog_gui.py:1487 iw/partition_gui.py:616
#: iw/upgrade_swap_gui.py:138 textw/upgrade_text.py:127
msgid "Mount Point"
msgstr "Csatoló pont"

#: iw/lvm_dialog_gui.py:1490 iw/partition_gui.py:615
msgid "Size (MB)"
msgstr "Méret (MB)"

#: iw/lvm_dialog_gui.py:1504 iw/osbootwidget.py:96
msgid "_Add"
msgstr "_Hozzáadás"

#: iw/lvm_dialog_gui.py:1507 iw/osbootwidget.py:100 iw/partition_gui.py:1859
#: iw/partition_gui.py:1874
msgid "_Edit"
msgstr "Sz_erkesztés"

#: iw/lvm_dialog_gui.py:1522
msgid "_Logical Volumes"
msgstr "_Logikai kötetek"

#: iw/network_gui.py:81 iw/network_gui.py:87
msgid "Error with Hostname"
msgstr "Hibás gazda név"

#: iw/network_gui.py:82
msgid "You must enter a valid hostname for this computer."
msgstr "Érvényes gazda nevet kell adni ennek a számítógépnek."

#: iw/network_gui.py:88
#, python-format
msgid ""
"The hostname \"%(hostname)s\" is not valid for the following reason:\n"
"\n"
"%(herrors)s"
msgstr ""

#: iw/network_gui.py:137
msgid "Select network interface"
msgstr ""

#: iw/network_gui.py:144
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please configure a network interface."
msgstr ""
"Ehhez szükséges, hogy a telepítés alatt legyen aktív hálózat kapcsolat.  "
"Kérem állítson be egy hálózat csatolót."

#: iw/osbootwidget.py:50
msgid "Boot loader operating system list"
msgstr "Rendszerindító operációs rendszer lista"

#: iw/osbootwidget.py:66
msgid "Default"
msgstr "Alapértelmezés"

#: iw/osbootwidget.py:66 iw/partition_gui.py:614
msgid "Label"
msgstr "Címke"

#: iw/osbootwidget.py:130
msgid "Add/Edit boot loader record"
msgstr ""

#: iw/osbootwidget.py:137
msgid ""
"Enter a label for the boot loader menu to display. The device (or hard drive "
"and partition number) is the device from which it boots."
msgstr ""
"Adja meg a rendszerindító menüben megjelenítendő címkét. A megadott "
"eszközről (vagy merevlemez és partíció számról) indul majd a rendszer."

#: iw/osbootwidget.py:145
msgid "_Label"
msgstr "_Címke"

#: iw/osbootwidget.py:153
msgid "_Device"
msgstr "Esz_köz"

#: iw/osbootwidget.py:183
msgid "Default Boot _Target"
msgstr "Alapértelme_zett rendszerindító cél"

#: iw/osbootwidget.py:212
msgid "You must specify a label for the entry"
msgstr "Meg kell adni egy címkét a bejegyzéshez"

#: iw/osbootwidget.py:222
msgid "Boot label contains illegal characters"
msgstr "A rendszerindító címke tiltott karaktereket tartalmaz"

#: iw/osbootwidget.py:247
msgid "Duplicate Label"
msgstr "Ismételt címke"

#: iw/osbootwidget.py:248
msgid "This label is already in use for another boot entry."
msgstr "Ezt a címkét egy másik rendszerindító bejegyzés már használja."

#: iw/osbootwidget.py:262
msgid "Duplicate Device"
msgstr "Ismételt eszköz"

#: iw/osbootwidget.py:263
msgid "This device is already being used for another boot entry."
msgstr "Ezt az eszközt egy másik rendszerindító bejegyzés már használja."

#: iw/osbootwidget.py:326
msgid "Cannot Delete"
msgstr "Nem lehet törölni"

#: iw/osbootwidget.py:327
#, python-format
msgid ""
"This boot target cannot be deleted because it is for the %s system you are "
"about to install."
msgstr ""
"Ez a rendszerindító bejegyzés nem törölhető, mert a most telepítendő %s "
"rendszerhez tartozik."

#: iw/partition_dialog_gui.py:58
msgid "Additional Size Options"
msgstr "További méret lehetőségek"

#: iw/partition_dialog_gui.py:63
msgid "_Fixed size"
msgstr "_Rögzített méret"

#: iw/partition_dialog_gui.py:65
msgid "Fill all space _up to (MB):"
msgstr "Összes hely kitöltése _eddig (MB):"

#: iw/partition_dialog_gui.py:72
msgid "Fill to maximum _allowable size"
msgstr "Kitöltés a max. _megengedett méretig"

#: iw/partition_dialog_gui.py:193
msgid "Error With Request"
msgstr ""

#: iw/partition_dialog_gui.py:405
msgid "Add Partition"
msgstr "Partició hozzáadása"

#: iw/partition_dialog_gui.py:407
#, python-format
msgid "Edit Partition: %s"
msgstr "%s partíció szerkesztése"

#: iw/partition_dialog_gui.py:444
msgid "File System _Type:"
msgstr "Fájl rendszer _típusa:"

#: iw/partition_dialog_gui.py:460
msgid "Allowable _Drives:"
msgstr "Engedélyezett e_szközök:"

#: iw/partition_dialog_gui.py:541
msgid "Force to be a _primary partition"
msgstr "Kényszerített első_dleges rész legyen"

#: iw/partition_gui.py:359
#, python-format
msgid "Drive %(drive)s (%(size)-0.f MB) (Model: %(model)s)"
msgstr ""

#: iw/partition_gui.py:399 iw/partition_gui.py:448 iw/partition_gui.py:515
#: iw/partition_gui.py:1000 iw/partition_gui.py:1079
msgid "Free"
msgstr "Szabad"

#: iw/partition_gui.py:480
#, python-format
msgid "LVM Volume Group %s (%-0.f MB)"
msgstr ""

#: iw/partition_gui.py:546
#, python-format
msgid "MD RAID ARRAY %s (%-0.f MB)"
msgstr ""

#: iw/partition_gui.py:617
msgid "Type"
msgstr "Típus"

#: iw/partition_gui.py:618 storage/__init__.py:1905
msgid "Format"
msgstr "Formázás"

#: iw/partition_gui.py:655
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr ""
"Csatolási pont/\n"
"RAID/Kötet"

#: iw/partition_gui.py:657
msgid ""
"Size\n"
"(MB)"
msgstr ""
"Méret\n"
"(MB)"

#: iw/partition_gui.py:762
msgid "Partitioning"
msgstr "Partícionálás"

#: iw/partition_gui.py:852
msgid ""
"The partitioning scheme you requested caused the following critical errors."
msgstr "A kért felosztási séma a következő kritikus hibákat okozta:"

#: iw/partition_gui.py:854
#, python-format
msgid ""
"You must correct these errors before you continue your installation of %s."
msgstr "E hibákat ki kell javítani, mielőtt folytatná %s telepítését."

#: iw/partition_gui.py:860
msgid "Partitioning Errors"
msgstr "Partícionálási hibák"

#: iw/partition_gui.py:867
msgid "The partitioning scheme you requested generated the following warnings."
msgstr "A kért felosztási séma a következő figyelmeztetéseket idézte elő:"

#: iw/partition_gui.py:869
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr "Tovább szeretne lépni a kért felosztási sémával?"

#: iw/partition_gui.py:874
msgid "Partitioning Warnings"
msgstr "Partícionálási figyelmeztetések"

#: iw/partition_gui.py:883
msgid ""
"The following pre-existing devices have been selected to be formatted, "
"destroying all data."
msgstr ""

#: iw/partition_gui.py:895
msgid "Format Warnings"
msgstr "Formázási figyelmeztetések"

#: iw/partition_gui.py:988
msgid "LVM Volume Groups"
msgstr "LVM kötet csoportok"

#: iw/partition_gui.py:1010
msgid "RAID Devices"
msgstr "RAID eszközök"

#: iw/partition_gui.py:1026 loader/hdinstall.c:214
msgid "Hard Drives"
msgstr "Merevlemezek"

#: iw/partition_gui.py:1083
msgid "Extended"
msgstr "Kibővített"

#: iw/partition_gui.py:1375
msgid "Cannot perform any creation action"
msgstr ""

#: iw/partition_gui.py:1376
msgid ""
"Note that the creation action requires one of the following:\n"
"\n"
"* Free space in one of the Hard Drives.\n"
"* At least two free Software RAID partitions.\n"
"* At least one free physical volume (LVM) partition.\n"
"* At least one Volume Group with free space."
msgstr ""

#: iw/partition_gui.py:1479
#, python-format
msgid ""
"Software RAID allows you to combine several disks into a larger RAID "
"device.  A RAID device can be configured to provide additional speed and "
"reliability compared to using an individual drive.  For more information on "
"using RAID devices please consult the %s documentation.\n"
msgstr ""

#: iw/partition_gui.py:1485
msgid ""
"To use RAID you must first create at least two partitions of type 'software "
"RAID'.  Then you can create a RAID device that can be formatted and "
"mounted.\n"
"\n"
msgstr ""
"RAID használatához először készíteni kell legalább két 'szoftveres RAID' "
"típusú részt.  Majd egy RAID eszközt, ami kialakítható és becsatolható.\n"
"\n"

#: iw/partition_gui.py:1489
#, python-format
msgid "You currently have %d software RAID partition free to use."
msgid_plural "You currently have %d software RAID partitions free to use."
msgstr[0] ""
msgstr[1] ""

#: iw/partition_gui.py:1493
msgid "About RAID"
msgstr ""

#: iw/partition_gui.py:1499
#, python-format
msgid ""
"Logical Volume Manager (LVM) is a 3 level construct. The fist level is made "
"up of disks or partitions formated with LVM metadata called Physical Volumes "
"(PV).  A Volume Group (VG) sits on top of one or more PVs. The VG, in turn, "
"is the base to creat one ore more Logical Volumes (LV).  Note that a VG can "
"be an aggregate of PVs from multiple physical disk.  For more information on "
"using LVM please consult the %s documentation\n"
msgstr ""

#: iw/partition_gui.py:1507
msgid ""
"To create a PV you need a partition with free space.  To create a VG you "
"need a PV that is not part of any existing VG.  To create a LV you need a VG "
"with free space.\n"
"\n"
msgstr ""

#: iw/partition_gui.py:1511
#, python-format
msgid "You currently have %d available PV free to use.\n"
msgid_plural "You currently have %d available PVs free to use.\n"
msgstr[0] ""
msgstr[1] ""

#: iw/partition_gui.py:1515
msgid "About LVM"
msgstr ""

#: iw/partition_gui.py:1616
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr "Nem sikerült lefoglalni a kért partíciókat: %s."

#: iw/partition_gui.py:1625
#, python-format
msgid "Warning: %s."
msgstr "Figyelmeztetés: %s."

#: iw/partition_gui.py:1656
msgid "Unable To Edit"
msgstr "Nem lehet szerkeszteni"

#: iw/partition_gui.py:1657
#, python-format
msgid ""
"You cannot edit this device:\n"
"\n"
"%s"
msgstr ""
"Ezen rész nem szerkeszthető:\n"
"\n"
"%s"

#: iw/partition_gui.py:1858
msgid "_Create"
msgstr ""

#: iw/partition_gui.py:1861
msgid "Re_set"
msgstr "_Visszaállítás"

#: iw/partition_gui.py:1889
msgid "Please Select A Device"
msgstr ""

#: iw/partition_ui_helpers_gui.py:110 iw/partition_ui_helpers_gui.py:132
#: iw/partition_ui_helpers_gui.py:134
msgid "<Not Applicable>"
msgstr "<Nem jellemző>"

#: iw/partition_ui_helpers_gui.py:314
msgid "_Format as:"
msgstr "_Formázás mint:"

#: iw/partition_ui_helpers_gui.py:336
msgid "Mi_grate filesystem to:"
msgstr "Fájlrendszer mi_grálása:"

#: iw/partition_ui_helpers_gui.py:366
msgid "_Resize"
msgstr "Át_méretezés"

#: iw/partition_ui_helpers_gui.py:421
#, python-format
msgid ""
"Partitions of type '%s' must be constrained to a single drive.  To do this, "
"select the drive in the 'Allowable Drives' checklist."
msgstr ""
"„%s” típusú részeket csak egyetlen eszközhöz lehet rendelni.  Az eszközt a "
"„Használható eszközök” listából lehet kiválasztani."

#: iw/raid_dialog_gui.py:388
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""
"Legalább két szoftveres RAID rész szükséges RAID eszköz készítéséhez.\n"
"\n"
"Először csináljon legalább két „szoftver RAID” részt, majd válassza újból a "
"„RAID” opciót."

#: iw/raid_dialog_gui.py:402
msgid "Make RAID Device"
msgstr "RAID eszköz készítése"

#: iw/raid_dialog_gui.py:405
#, python-format
msgid "Edit RAID Device: %s"
msgstr "%s RAID eszköz szerkesztése"

#: iw/raid_dialog_gui.py:407
msgid "Edit RAID Device"
msgstr "RAID eszköz szerkesztése"

#: iw/raid_dialog_gui.py:474
msgid "RAID _Device:"
msgstr "RAID _eszköz:"

#: iw/raid_dialog_gui.py:492
msgid "RAID _Level:"
msgstr "RAID _szint:"

#: iw/raid_dialog_gui.py:518
msgid "_RAID Members:"
msgstr "_RAID tagok:"

#: iw/raid_dialog_gui.py:535
msgid "Number of _spares:"
msgstr "Tartalékok _száma:"

#: iw/raid_dialog_gui.py:545
msgid "_Format partition?"
msgstr "Partíció _formázása?"

#: iw/task_gui.py:55
#, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a "
"missing repodata directory.  Please ensure that your repository has been "
"correctly generated.\n"
"\n"
"%s"
msgstr ""
"Nem lehet beolvasni a csomagleíró adatokat a tárolóból.  Talán hiányzik egy "
"repodata mappa.  Kérem ellenőrizze, hogy jól készítették-e el a tárolót.\n"
"\n"
"%s"

#: iw/task_gui.py:129
msgid "Edit Repository"
msgstr "Tároló szerkesztése"

#: iw/task_gui.py:153
#, python-format
msgid ""
"The repository %s has already been added.  Please choose a different "
"repository name and URL."
msgstr ""
"„%s” tárolót már hozzáadta. Kérem válasszon másik tároló nevet és URL-t."

#: iw/task_gui.py:239
msgid "Invalid Proxy URL"
msgstr "Érvénytelen proxy URL"

#: iw/task_gui.py:240
msgid "You must provide an HTTP, HTTPS, or FTP URL to a proxy."
msgstr "Meg kell adnia egy HTTP, HTTPS vagy FTP URL-t egy proxyhoz."

#: iw/task_gui.py:252 iw/task_gui.py:439
msgid "Invalid Repository URL"
msgstr "Érvénytelen tároló URL"

#: iw/task_gui.py:253 iw/task_gui.py:440
msgid "You must provide an HTTP, HTTPS, or FTP URL to a repository."
msgstr "Meg kell adnia egy HTTP, HTTPS vagy FTP URL-t egy tárolóhoz."

#: iw/task_gui.py:274 iw/task_gui.py:449
msgid "No Media Found"
msgstr "Nem találtam hordozót"

#: iw/task_gui.py:275 iw/task_gui.py:450
msgid ""
"No installation media was found. Please insert a disc into your drive and "
"try again."
msgstr ""
"Nem találtam telepítő hordozót. Kérem tegyen be egy lemezt, és próbálja újra."

#: iw/task_gui.py:306 iw/task_gui.py:473
msgid "Please enter an NFS server and path."
msgstr "Kérem adjon meg NFS kiszolgáló nevet és utat."

#: iw/task_gui.py:325
#, python-format
msgid ""
"The following error occurred while setting up the repository:\n"
"\n"
"%s"
msgstr ""
"A következő hiba merült fel a tároló beállítása közben:\n"
"\n"
"%s"

#: iw/task_gui.py:348
msgid "Invalid Repository Name"
msgstr "Érvénytelen tároló név"

#: iw/task_gui.py:349
msgid "You must provide a repository name."
msgstr "Meg kell adnia egy tároló nevet."

#: iw/task_gui.py:505 ui/addrepo.glade.h:3
msgid "Add Repository"
msgstr "Tároló hozzáadása"

#: iw/task_gui.py:510
msgid "No Software Repos Enabled"
msgstr "Nincs engedélyezett szoftver tároló"

#: iw/task_gui.py:511
msgid ""
"You must have at least one software repository enabled to continue "
"installation."
msgstr ""
"Legalább egy szoftver tárolót engedélyezni kell, hogy folytassuk a "
"telepítést."

#: iw/timezone_gui.py:63 textw/timezone_text.py:95
msgid "Time Zone Selection"
msgstr "Időzóna választás"

#: iw/upgrade_bootloader_gui.py:37 textw/upgrade_bootloader_text.py:130
msgid "Upgrade Boot Loader Configuration"
msgstr "Rendszerbetöltő beállítások frissítése"

#: iw/upgrade_bootloader_gui.py:123
msgid "_Update boot loader configuration"
msgstr "Rendszerbetöltő beállításainak _frissítése"

#: iw/upgrade_bootloader_gui.py:124
msgid "This will update your current boot loader."
msgstr "Ez frissíti az aktuális rendszerbetöltőt."

#: iw/upgrade_bootloader_gui.py:128 textw/upgrade_bootloader_text.py:103
msgid ""
"Due to system changes, your boot loader configuration can not be "
"automatically updated."
msgstr ""
"A rendszerben történt módosítások miatt a rendszerindító beállításai nem "
"frissíthetők automatikusan."

#: iw/upgrade_bootloader_gui.py:131 textw/upgrade_bootloader_text.py:107
msgid ""
"The installer is unable to detect the boot loader currently in use on your "
"system."
msgstr "A telepítő nem képes érzékelni a most használt rendszerindítót."

#: iw/upgrade_bootloader_gui.py:138 textw/upgrade_bootloader_text.py:117
#, python-format
msgid ""
"The installer has detected the %(type)s boot loader currently installed on %"
"(bootDev)s."
msgstr ""

#: iw/upgrade_bootloader_gui.py:142
msgid "This is the recommended option."
msgstr "Ez a javasolt beállítás."

#: iw/upgrade_bootloader_gui.py:147
msgid "_Create new boot loader configuration"
msgstr "Ú_j rendszerindító beállítás készítése"

#: iw/upgrade_bootloader_gui.py:149
msgid ""
"This option creates a new boot loader configuration.  If you wish to switch "
"boot loaders, you should choose this."
msgstr ""
"Ezzel új rendszerindító beállítást lehet készíteni.  Ha le szeretné cserélni "
"az eddigi rendszerindítót, válassza ezt."

#: iw/upgrade_bootloader_gui.py:156
msgid "_Skip boot loader updating"
msgstr "Rendszerbetöltő frissítés ki_hagyása"

#: iw/upgrade_bootloader_gui.py:157
msgid ""
"This option makes no changes to boot loader configuration.  If you are using "
"a third party boot loader, you should choose this."
msgstr ""
"Ebben az esetben nem változik meg a rendszerindító beállítás.  Ha külső "
"forrásból származó rendszerindítót használ, akkor válassza ezt a lehetőséget."

#: iw/upgrade_bootloader_gui.py:168
msgid "What would you like to do?"
msgstr "Mit szeretne tenni?"

#: iw/upgrade_migratefs_gui.py:38 textw/upgrade_text.py:39
msgid "Migrate File Systems"
msgstr "Fájlrendszerek migrálása"

#: iw/upgrade_migratefs_gui.py:69
#, python-format
msgid ""
"This release of %(productName)s supports an updated file system, which has "
"several benefits over the file system traditionally shipped in %(productName)"
"s.  This installation program can migrate formatted partitions without data "
"loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""

#: iw/upgrade_swap_gui.py:35
msgid "Upgrade Swap Partition"
msgstr "Lapozó rész frissítése"

#: iw/upgrade_swap_gui.py:92 textw/upgrade_text.py:110
#, python-format
msgid ""
"Recent kernels (2.4 or newer) need significantly more swap than older "
"kernels, up to twice the amount of RAM on the system.  You currently have %"
"dMB of swap configured, but you may create additional swap space on one of "
"your file systems now."
msgstr ""
"A 2.4-es és újabb Linux rendszermagok nagyobb lapozó területet igényelnek, "
"mint a régebbi változatok - ez elérheti a RAM méretének kétszeresét.  "
"Jelenleg %d MB méretű lapozó van beállítva, de most hozzáadhat további "
"lapozó helyet is valamelyik fájl rendszeren."

#: iw/upgrade_swap_gui.py:99
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""
"\n"
"\n"
"A telepítő %s MB RAM-ot talált a gépben.\n"

#: iw/upgrade_swap_gui.py:111
msgid "I _want to create a swap file"
msgstr "Lapo_zó fájlt szeretnék csinálni"

#: iw/upgrade_swap_gui.py:120
msgid "Select the _partition to put the swap file on:"
msgstr "Erre a _partícióra kerüljön a lapozó fájl:"

#: iw/upgrade_swap_gui.py:138 textw/upgrade_text.py:128
msgid "Partition"
msgstr "Partíció"

#: iw/upgrade_swap_gui.py:138
msgid "Free Space (MB)"
msgstr "Szabad hely (MB)"

#: iw/upgrade_swap_gui.py:156
#, python-format
msgid ""
"A minimum swap file size of %d MB is recommended.  Please enter a size for "
"the swap file:"
msgstr ""
"A lapozó fájlt érdemes legalább %d MB-ra venni.  Kérem adja meg a lapozó "
"fájl méretét:"

#: iw/upgrade_swap_gui.py:171
msgid "Swap file _size (MB):"
msgstr "Lap_ozó fájl mérete (MB):"

#: iw/upgrade_swap_gui.py:181
msgid "I _don't want to create a swap file"
msgstr "Nem a_karok lapozó fájlt készíteni"

#: iw/upgrade_swap_gui.py:191
msgid ""
"A swap file is strongly recommended. Failure to create one could cause the "
"installer to abort abnormally.  Are you sure you wish to continue?"
msgstr ""
"Mindenképpen azt ajánljuk, hogy készítsen egy lapozó fájlt. Ha ezt nem teszi "
"meg, előfordulhat, hogy a telepítő váratlanul leáll.  Biztosan folytatni "
"kívánja?"

#: iw/upgrade_swap_gui.py:199 textw/upgrade_text.py:197
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr "A lapozó fájl méretének 1 és 2000 MB közé kell esnie."

#: iw/upgrade_swap_gui.py:206 textw/upgrade_text.py:192
msgid ""
"There is not enough space on the device you selected for the swap partition."
msgstr "Nincs elég hely a kiválasztott eszközön a lapozó rész elkészítéséhez."

#: iw/welcome_gui.py:48 textw/welcome_text.py:36
msgid "Network Install Required"
msgstr "Hálózati telepítés szükséges"

#: iw/welcome_gui.py:49 textw/welcome_text.py:37
msgid ""
"Your installation source is set to a network location, but no netork devices "
"were found on your system.  To avoid a network installation, boot with the "
"full DVD, full CD set, or do not pass a repo= parameter that specifies a "
"network source."
msgstr ""
"A telepítés forrását hálózati helyre állította, de nem találtam hálózati "
"eszközt a rendszerben.  A hálózati telepítést elkerülendő indítson újra egy "
"teljes DVD vagy CD készlettel, vagy ne adjon meg hálózati forrást jelölő "
"'repo=' paramétert."

#: iw/welcome_gui.py:59
msgid "E_xit Installer"
msgstr "_Kilépés a telepítőből"

#: iw/zipl_gui.py:37
msgid "z/IPL Boot Loader Configuration"
msgstr "z/IPL rendszerindító beállításai"

#: iw/zipl_gui.py:60
msgid "The z/IPL boot loader will be installed on your system."
msgstr "z/IPL rendszerindítót telepít a gépre."

#: iw/zipl_gui.py:62
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition "
"setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by "
"default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change "
"the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or "
"your setup may require."
msgstr ""
"A z/IPL rendszerindító telepítése következik.\n"
"\n"
"A gyökér rész a telepítés korábbi fázisában kijelölt rész lesz.\n"
"\n"
"A rendszer indítása a kiválasztott alapértelmezett rendszermaggal történik.\n"
"\n"
"Ha később, a telepítés után szeretne módosításokat végrehajtani, akkor "
"módosítsa az /etc/zipl.conf beállítás fájlt.\n"
"\n"
"Adja meg azokat a rendszermag-paramétereket, melyek a gép vagy a "
"konfiguráció használatához szükségesek."

#: iw/zipl_gui.py:89 textw/zipl_text.py:71
msgid "Kernel Parameters"
msgstr "Rendszermag paraméterek"

#: iw/zipl_gui.py:92 iw/zipl_gui.py:95
msgid "Chandev Parameters"
msgstr "Chandev paraméterek"

#: loader/net.h:27 textw/netconfig_text.py:164
msgid "Dynamic IP configuration (DHCP)"
msgstr ""

#: loader/net.h:28
msgid "Manual configuration"
msgstr ""

#: loader/net.h:30
msgid "Automatic, DHCP only"
msgstr ""

#: loader/net.h:31
msgid "Automatic"
msgstr ""

#: loader/cdinstall.c:188 loader/cdinstall.c:218 loader/mediacheck.c:60
msgid "Media Check"
msgstr "Adathordozó ellenőrzése"

#: loader/cdinstall.c:188 loader/cdinstall.c:191 loader/cdinstall.c:218
#: loader/cdinstall.c:226 loader/method.c:324
msgid "Test"
msgstr "Próba"

#: loader/cdinstall.c:188 loader/cdinstall.c:192
msgid "Eject Disc"
msgstr "Lemez kiadása"

#: loader/cdinstall.c:189
#, c-format
msgid ""
"Choose \"%s\" to test the disc currently in the drive, or \"%s\" to eject "
"the disc and insert another for testing."
msgstr ""
"Válassza „%s” opciót az olvasóban levő lemez ellenőrzéséhez, vagy „%s” "
"opciót a lemez kivételéhez, és újabb tesztelendő lemez behelyezéséhez."

#: loader/cdinstall.c:209
msgid "Media ejected"
msgstr ""

#: loader/cdinstall.c:210
#, c-format
msgid ""
"The disc currently inserted to your drive was ejected. Press %s to continue."
msgstr ""

#: loader/cdinstall.c:219
#, c-format
msgid ""
"If you would like to test additional media, insert the next disc and press "
"\"%s\". Testing each disc is not strictly required, however it is highly "
"recommended.  Minimally, the discs should be tested prior to using them for "
"the first time. After they have been successfully tested, it is not required "
"to retest each disc prior to using it again."
msgstr ""
"Amennyiben további adathordozót is akar még ellenőrizni, helyezze be a "
"következő lemezt, és nyomjon \"%s\" gombot. Minden egyes lemez ellenőrzése "
"nem feltétlenül szükséges, azonban ajánlatos. Legalább az első használat "
"előtt érdemes a lemezeket ellenőrizni. Amennyiben már sikeresen ellenőrizték "
"őket, nem szükséges az újra ellenőrizni."

#: loader/cdinstall.c:241
#, c-format
msgid ""
"The %s disc was not found in any of your drives. Please insert the %s disc "
"and press %s to retry."
msgstr ""
"%s lemez nincs egyik olvasóban sem. Kérem tegye be %s lemezt, és nyomjon %s "
"gombot, hogy újra próbálja."

#: loader/cdinstall.c:260
msgid "Disc Found"
msgstr "Lemez felismerve"

#: loader/cdinstall.c:261
#, c-format
msgid ""
"To begin testing the media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""
"Nyomjon %s gombot, hogy leellenőrizzem az adathordozót.\n"
"\n"
"Válassza %s gombot, hogy kihagyjam az adathordozó ellenőrzését, és "
"megkezdjem a telepítést."

#: loader/cdinstall.c:347
msgid "Scanning"
msgstr "Beolvasás"

#: loader/cdinstall.c:347
#, c-format
msgid "Looking for installation images on CD device %s\n"
msgstr "Telepítő képeket keresek %s CD eszközön\n"

#: loader/cdinstall.c:349
#, c-format
msgid "Looking for installation images on CD device %s"
msgstr "Telepítő képeket keresek %s CD eszközön"

#: loader/cdinstall.c:429
#, c-format
msgid ""
"The %s disc was not found in any of your CDROM drives. Please insert the %s "
"disc and press %s to retry."
msgstr ""
"Egyik CD-ROM olvasóban sincs %s lemez. Kérem tegyen %s lemezt, és nyomjon %s "
"gombot, hogy újra próbálja."

#: loader/cdinstall.c:438
msgid "Disc Not Found"
msgstr "Lemez nem található"

#: loader/cdinstall.c:439 loader/driverdisk.c:479 loader/driverdisk.c:515
#: loader/driverdisk.c:602 loader/driverselect.c:78 loader/driverselect.c:152
#: loader/driverselect.c:178 loader/hdinstall.c:214 loader/hdinstall.c:268
#: loader/kbd.c:119 loader/loader.c:520 loader/loader.c:537
#: loader/loader.c:1496 loader/loader.c:1529 loader/net.c:607 loader/net.c:991
#: loader/net.c:2164 loader/net.c:2183 loader/nfsinstall.c:91
#: loader/urls.c:268 storage/__init__.py:165 storage/devicetree.py:93
#: textw/constants_text.py:52
msgid "Back"
msgstr "Vissza"

#: loader/cdinstall.c:513
msgid "Cannot find kickstart file on CDROM."
msgstr "Nem találtam kickstart fájlt a CD-n."

#: loader/copy.c:51 loader/method.c:278
#, c-format
msgid "Failed to read directory %s: %m"
msgstr "%s mappa beolvasása sikertelen: %m"

#: loader/driverdisk.c:339
msgid "Loading"
msgstr "Betöltés"

#: loader/driverdisk.c:339
msgid "Reading driver disk"
msgstr "Eszközkezelő szoftver lemezének olvasása"

#: loader/driverdisk.c:473 loader/driverdisk.c:510
msgid "Driver Disk Source"
msgstr "Eszközkezelő szoftver lemezének forrása"

#: loader/driverdisk.c:474
msgid ""
"You have multiple devices which could serve as sources for a driver disk.  "
"Which would you like to use?"
msgstr ""
"Több olyan meghajtó található a gépben, amely forrásául szolgálhat az "
"eszközkezelő szoftver lemezének.  Melyiket szeretné használni?"

#: loader/driverdisk.c:511
msgid ""
"There are multiple partitions on this device which could contain the driver "
"disk image.  Which would you like to use?"
msgstr ""
"Több olyan partíció található ezen az eszközön, amely tartalmazhatja a "
"eszközkezelő szoftver lemezének képmását.  Melyiket szeretné használni?"

#: loader/driverdisk.c:546
msgid "Failed to mount partition."
msgstr "Nem sikerült becsatolni a partíciót."

#: loader/driverdisk.c:557
msgid "Select driver disk image"
msgstr "Válasszon eszközkezelő szoftver lemez képmását"

#: loader/driverdisk.c:558
msgid "Select the file which is your driver disk image."
msgstr ""
"Válassza ki a fájlt, amelyik a eszközkezelő szoftver lemezének képmása."

#: loader/driverdisk.c:587
msgid "Failed to load driver disk from file."
msgstr "Nem sikerült betölteni a vezérlő képmást a fájlból."

#: loader/driverdisk.c:599
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr ""
"Tegye be a vezérlőprogram lemezt /dev/%s olvasóba, majd nyomjon „OK”-t, hogy "
"folytassam."

#: loader/driverdisk.c:602
msgid "Insert Driver Disk"
msgstr "Tegyen be eszközkezelő szoftver lemezét"

#: loader/driverdisk.c:615
msgid "Failed to mount driver disk."
msgstr "Nem sikerült becsatolni a eszközkezelő szoftver lemezét."

#: loader/driverdisk.c:623
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr "%s jelen kiadásában érvénytelen eszközkezelő szoftver lemeze."

#: loader/driverdisk.c:694
msgid "Manually choose"
msgstr "Kézi kiválasztás"

#: loader/driverdisk.c:695
msgid "Load another disk"
msgstr "Másik lemez betöltése"

#: loader/driverdisk.c:696
msgid ""
"No new drivers were found on this driver disk.This may indicate that this "
"disk has already been loaded,or that the drivers it contains don't match "
"your hardware.Would you like to manually select the driver, continue anyway, "
"or load another driver disk?"
msgstr ""

#: loader/driverdisk.c:740
msgid "Driver disk"
msgstr "Vezérlőprogram lemez"

#: loader/driverdisk.c:741
msgid "Do you have a driver disk?"
msgstr "Van vezérlőprogram lemeze?"

#: loader/driverdisk.c:750
msgid "More Driver Disks?"
msgstr "Van még vezérlőprogram lemez?"

#: loader/driverdisk.c:751
msgid "Do you wish to load any more driver disks?"
msgstr "Szeretne további vezérlő lemezeket betölteni?"

#: loader/driverdisk.c:802 loader/driverdisk.c:840 loader/hdinstall.c:362
#: loader/kickstart.c:138 loader/kickstart.c:148 loader/kickstart.c:190
#: loader/kickstart.c:195 loader/kickstart.c:524 loader/modules.c:396
#: loader/modules.c:412 loader/net.c:1796 loader/net.c:1820
#: loader/nfsinstall.c:384 loader/urlinstall.c:374 loader/urlinstall.c:385
#: loader/urlinstall.c:392
msgid "Kickstart Error"
msgstr "Kickstart hiba"

#: loader/driverdisk.c:803
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr "Ismeretlen vezérlő lemez kickstart forrás: %s"

#: loader/driverdisk.c:841
#, c-format
msgid ""
"The following invalid argument was specified for the kickstart driver disk "
"command: %s"
msgstr ""

#: loader/driverselect.c:67
#, c-format
msgid ""
"Please enter any parameters which you wish to pass to the %s module "
"separated by spaces.  If you don't know what parameters to supply, skip this "
"screen by pressing the \"OK\" button."
msgstr ""
"Kérem adja meg %s modulnak átadni kívánt paramétereket, szóközzel "
"elválasztva.  Ha nem tudja, milyen paraméterekre van szükség, lépjen tovább "
"a következő képernyőre „OK” gombot nyomva."

#: loader/driverselect.c:88
msgid "Enter Module Parameters"
msgstr "Modul paraméterek megadása"

#: loader/driverselect.c:151
msgid "No drivers found"
msgstr "Nincs vezérlőprogram"

#: loader/driverselect.c:151
msgid "Load driver disk"
msgstr "Vezérlő lemez betöltése"

#: loader/driverselect.c:152
msgid ""
"No drivers were found to manually insert.  Would you like to use a driver "
"disk?"
msgstr ""
"Nincs kézzel beszúrható vezérlő.  Szeretne vezérlőprogram lemezt használni?"

#: loader/driverselect.c:170
msgid ""
"Please select the driver below which you wish to load.  If it does not "
"appear and you have a driver disk, press F2."
msgstr ""
"Kérem válassza ki alább a betöltendő vezérlőprogramot.  Ha nem jelenik meg, "
"és rendelkezik vezérlő lemez, nyomjon F2-t."

#: loader/driverselect.c:179
msgid "Specify optional module arguments"
msgstr "Választható modul argumentumok megadása"

#: loader/driverselect.c:204
msgid "Select Device Driver to Load"
msgstr "Válassza ki a betöltendő vezérlőprogramot"

#: loader/hdinstall.c:118
msgid ""
"An error occured finding the installation image on your hard drive.  Please "
"check your images and try again."
msgstr ""
"Hiba történt a telepítő képet keresése közben a merevlemezen.  Kérem "
"ellenőrizze a képeket, és próbálkozzon újra."

#: loader/hdinstall.c:215
msgid ""
"You don't seem to have any hard drives on your system! Would you like to "
"configure additional devices?"
msgstr ""
"Úgy tűnik, nincs egyetlen merevlemez vezérlő sem a rendszerben. Kíván "
"további eszközöket beállítani?"

#: loader/hdinstall.c:228
#, c-format
msgid ""
"What partition and directory on that partition holds the installation image "
"for %s?  If you don't see the disk drive you're using listed here, press F2 "
"to configure additional devices."
msgstr ""
"Melyik részen, és azon belül melyik mappában van a telepítő képmás a %s "
"rendszerhez?  Ha nem látja a listában a használt lemezt, nyomjon F2-t "
"további eszközök beállításához."

#: loader/hdinstall.c:252
msgid "Directory holding image:"
msgstr "Képeket tartalmazó mappa:"

#: loader/hdinstall.c:280
msgid "Select Partition"
msgstr "Válasszon partíciót"

#: loader/hdinstall.c:328
#, c-format
msgid "Device %s does not appear to contain an installation image."
msgstr "Úgy tűnik %s eszköz nem tartalmaz telepítő képet."

#: loader/hdinstall.c:363
#, c-format
msgid "Bad argument to HD kickstart method command: %s"
msgstr ""

#: loader/hdinstall.c:443 loader/hdinstall.c:506
msgid "Cannot find kickstart file on hard drive."
msgstr "Nincs kickstart fájl a merevlemezen."

#: loader/hdinstall.c:493
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr "Nincs merevlemez %s BIOS lemezhez"

#: loader/kbd.c:117
msgid "Keyboard Type"
msgstr "Billentyűzet típus"

#: loader/kbd.c:118
msgid "What type of keyboard do you have?"
msgstr "Milyen típusú billentyűzete van?"

#: loader/kickstart.c:139
#, c-format
msgid "Error opening kickstart file %s: %m"
msgstr "%s kickstart fájlt megnyitása közben hiba történt: %m"

#: loader/kickstart.c:149
#, c-format
msgid "Error reading contents of kickstart file %s: %m"
msgstr "%s kickstart fájlt beolvasása közben hiba történt: %m"

#: loader/kickstart.c:191
#, c-format
msgid "Error in %s on line %d of kickstart file %s."
msgstr "%s hibás: %d. sor %s kickstart fájl."

#: loader/kickstart.c:196
#, c-format
msgid "Missing options on line %d of kickstart file %s."
msgstr ""

#: loader/kickstart.c:303
msgid "Cannot find ks.cfg on removable media."
msgstr "Nincs ks.cfg fájl a cserélhető adathordozón."

#: loader/kickstart.c:339
msgid ""
"Unable to download the kickstart file.  Please modify the kickstart "
"parameter below or press Cancel to proceed as an interactive installation."
msgstr ""
"Nem sikerült letölteni a Kickstart-fájlt.  Kérem változtassa meg az alábbi "
"kickstart paramétert, vagy nyomjon Mégsem-et, hogy interaktívan folytassuk a "
"telepítést."

#: loader/kickstart.c:348
msgid "Error downloading kickstart file"
msgstr "Kickstart fájlt letöltése hibás"

#: loader/kickstart.c:525
#, c-format
msgid "Bad argument to shutdown kickstart method command: %s"
msgstr ""

#: loader/lang.c:63 loader/loader.c:233
#, c-format
msgid "Welcome to %s for %s"
msgstr "Üdvözli %s %s"

#: loader/lang.c:64
#, c-format
msgid "Welcome to %s for %s - Rescue Mode"
msgstr "Üdvözöli %s %s – Mentő mód"

#: loader/lang.c:65 loader/loader.c:257
msgid ""
"  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr ""
"  <Tab>/<AltTab> elemek között  |  <Szóköz> megjelölés  |  <F12> köv. "
"képernyő  "

#: loader/lang.c:370
msgid "Choose a Language"
msgstr "Válasszon nyelvet"

#: loader/loader.c:136
msgid "Local CD/DVD"
msgstr "Helyi CD/DVD"

#: loader/loader.c:137
msgid "Hard drive"
msgstr "Merevlemez"

#: loader/loader.c:138
msgid "NFS directory"
msgstr "NFS mappa"

#: loader/loader.c:474 loader/loader.c:515
msgid "Update Disk Source"
msgstr "Frissítő lemez forrása"

#: loader/loader.c:475
msgid ""
"You have multiple devices which could serve as sources for an update disk.  "
"Which would you like to use?"
msgstr ""
"Több eszköz van a rendszerben, ami tartalmazhat frissítő lemezt. Melyiket "
"szeretné használni?"

#: loader/loader.c:516
msgid ""
"There are multiple partitions on this device which could contain the update "
"disk image.  Which would you like to use?"
msgstr ""
"Több rész van ezen az eszközön, ami tartalmazhat frissítő lemez képmást.  "
"Melyiket szeretné használni?"

#: loader/loader.c:534
#, c-format
msgid "Insert your updates disk into %s and press \"OK\" to continue."
msgstr ""
"Tegye be a frissítés lemezt %s eszközbe, majd nyomjon „OK”-t, hogy "
"folytassam."

#: loader/loader.c:537
msgid "Updates Disk"
msgstr "Frissítő lemez"

#: loader/loader.c:555
msgid "Failed to mount updates disk"
msgstr "Nem sikerült becsatolni a frissítő lemezt."

#: loader/loader.c:560
msgid "Updates"
msgstr "Frissítések"

#: loader/loader.c:560
msgid "Reading anaconda updates"
msgstr "Anaconda frissítések olvasása"

#: loader/loader.c:596
msgid ""
"Unable to download the updates image.  Please modify the updates location "
"below or press Cancel to proceed without updates.."
msgstr ""
"Nem sikerült letölteni a frissítő képmást.  Kérem módosítsa alább a "
"frissítések helyét, vagy nyomjon Mégsem-et, hogy frissítések nélkül "
"folytatódjon a telepítés..."

#: loader/loader.c:605
msgid "Error downloading updates image"
msgstr "Hiba a frissítő képmás letöltése közben"

#: loader/loader.c:1248
#, c-format
msgid "You do not have enough RAM to install %s on this machine."
msgstr "Nincs elég memória ebben a gépben %s telepítéséhez."

#: loader/loader.c:1281
msgid "Unsupported Hardware Detected"
msgstr ""

#: loader/loader.c:1282
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat.  For "
"more information on supported hardware, please refer to http://www.redhat."
"com/hardware."
msgstr ""

#: loader/loader.c:1348
msgid "Media Detected"
msgstr "Adathordozó érzékelve"

#: loader/loader.c:1349
msgid "Found local installation media"
msgstr "Helyi telepítő adathordozó érzékelve"

#: loader/loader.c:1488
msgid "Rescue Method"
msgstr "Mentő eljárás"

#: loader/loader.c:1489
msgid "Installation Method"
msgstr "Telepítő eljárás"

#: loader/loader.c:1491
msgid "What type of media contains the rescue image?"
msgstr "Milyen típusú hordozón van a mentő képmás?"

#: loader/loader.c:1493
msgid "What type of media contains the installation image?"
msgstr "Milyen típusú hordozón van a telepítő képmás?"

#: loader/loader.c:1528
msgid "No driver found"
msgstr "Nincs vezérlő"

#: loader/loader.c:1528
msgid "Select driver"
msgstr "Válasszon vezérlőt"

#: loader/loader.c:1529
msgid "Use a driver disk"
msgstr "Vezérlő lemez használata"

#: loader/loader.c:1530
msgid ""
"Unable to find any devices of the type needed for this installation type.  "
"Would you like to manually select your driver or use a driver disk?"
msgstr ""
"Nem találtam a telepítés típusának megfelelő eszközt.  Szeretne vezérlőt "
"választani, vagy vezérlő lemezt használni?"

#: loader/loader.c:1762
msgid "The following devices have been found on your system."
msgstr "A következő eszközöket találtam a rendszerben."

#: loader/loader.c:1764
msgid ""
"No device drivers have been loaded for your system.  Would you like to load "
"any now?"
msgstr ""
"Nincs eszköz vezérlő betöltve a rendszerbe.  Be szeretne most tölteni egyet?"

#: loader/loader.c:1768
msgid "Devices"
msgstr "Eszközök"

#: loader/loader.c:1769
msgid "Done"
msgstr "Kész"

#: loader/loader.c:1770
msgid "Add Device"
msgstr "Eszköz hozzáadása"

#: loader/loader.c:2060
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr "betöltő már lefutott.  Parancssor indítása.\n"

#: loader/loader.c:2207
#, c-format
msgid "Driver disc was detected in %s. Do you want to use it?."
msgstr ""

#: loader/loader.c:2211
msgid "Driver disc detected"
msgstr ""

#: loader/loader.c:2211
msgid "Use it"
msgstr ""

#: loader/loader.c:2211
msgid "Skip it"
msgstr ""

#: loader/loader.c:2577
#, c-format
msgid "Running anaconda %s, the %s rescue mode - please wait.\n"
msgstr "Anaconda %s, a %s mentő mód fut – kérem várjon.\n"

#: loader/loader.c:2579
#, c-format
msgid "Running anaconda %s, the %s system installer - please wait.\n"
msgstr "Anaconda %s, a %s rendszer telepítő fut – kérem várjon.\n"

#: loader/mediacheck.c:46
#, c-format
msgid "Unable to find install image %s"
msgstr "Nem találtam e telepítési képmást: %s"

#: loader/mediacheck.c:52
#, c-format
msgid "Checking \"%s\"."
msgstr "„%s” ellenőrzése."

#: loader/mediacheck.c:54
#, c-format
msgid "Checking media."
msgstr "Adathordozó ellenőrzése."

#: loader/mediacheck.c:86
msgid ""
"Unable to find the checksum in the image.  This probably means the disc was "
"created without adding the checksum."
msgstr ""

#: loader/mediacheck.c:93
msgid "Unable to open the image."
msgstr ""

#: loader/mediacheck.c:97
msgid ""
"The image which was just tested has errors. This could be due to a corrupt "
"download or a bad disc.  If applicable, please clean the disc and try "
"again.  If this test continues to fail you should not continue the install."
msgstr ""
"A tesztelt képmás hibás. Ezt okozhatta egy letöltési hiba vagy lemezhiba.  "
"Ha érdemes, tisztítsa meg a lemezt, és próbálja újra.  Ne folytassa a "
"telepítést, ha a teszt továbbra sem sikeres."

#: loader/mediacheck.c:105
msgid "Success"
msgstr "Sikerült"

#: loader/mediacheck.c:106
msgid ""
"The image which was just tested was successfully verified.  It should be OK "
"to install from this media.  Note that not all media/drive errors can be "
"detected by the media check."
msgstr ""
"A lemezképet – amit éppen kipróbált – sikeresen ellenőriztem.  Erről a "
"adathordozóról nyugodtan telepíthet.  Figyelem: nem tud minden hordozó/"
"olvasó hibát érzékelni a lemezellenőrzés."

#: loader/method.c:321
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""
"El szeretné végezni az ellenőrző összeg vizsgálatát ezen ISO képmásra:\n"
"\n"
"   %s?"

#: loader/method.c:324
msgid "Checksum Test"
msgstr "Ellenőrző összeg vizsgálata"

#: loader/modules.c:397
#, c-format
msgid "Bad argument to device kickstart method command: %s"
msgstr ""

#: loader/modules.c:413
msgid "A module name must be specified for the kickstart device command."
msgstr "Modul nevet is meg kell adni a kickstart device parancsában."

#: loader/net.c:112
msgid "Invalid Prefix"
msgstr "Érvénytelen előtag"

#: loader/net.c:113
msgid ""
"Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for "
"IPv6 networks"
msgstr ""
"Az előtag 1 és 32 közé kell essen IPv4 hálózatnál, vagy 1 és 128 közé IPv6 "
"hálózatnál."

#: loader/net.c:494 loader/net.c:555
#, c-format
msgid "There was an error configuring your network interface."
msgstr "Hálózat csatoló beállítása alatt hiba történt."

#: loader/net.c:496
#, c-format
msgid ""
"\n"
"This cannot be corrected in cmdline mode.\n"
"Halting.\n"
msgstr ""

#: loader/net.c:554 textw/netconfig_text.py:403
msgid "Network Error"
msgstr "Hálózat hiba"

#: loader/net.c:585
msgid "Enable IPv4 support"
msgstr "IPv4 támogatás engedélyezése"

#: loader/net.c:599
msgid "Enable IPv6 support"
msgstr "IPv6 támogatás engedélyezése"

#: loader/net.c:639
msgid "Configure TCP/IP"
msgstr "TCP/IP beállítás"

#: loader/net.c:696
msgid "Missing Protocol"
msgstr "Hiányzó protokoll"

#: loader/net.c:697
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr "Legalább egy protokollt (IPv4 vagy IPv6) ki kell választani."

#: loader/net.c:704
msgid "IPv4 Needed for NFS"
msgstr "NFS-hez IPv4 kell"

#: loader/net.c:705
msgid "NFS installation method requires IPv4 support."
msgstr "NFS telepítő eljárás IPv4 támogatást igényel."

#: loader/net.c:811
msgid "IPv4 address:"
msgstr "IPv4 cím:"

#: loader/net.c:823 loader/net.c:890
msgid "/"
msgstr "/"

#: loader/net.c:878
msgid "IPv6 address:"
msgstr "IPv6 cím:"

#: loader/net.c:944 textw/netconfig_text.py:184 textw/netconfig_text.py:224
msgid "Gateway:"
msgstr "Átjáró:"

#: loader/net.c:952
msgid "Name Server:"
msgstr "Névkiszolgáló:"

#: loader/net.c:997
msgid ""
"Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For "
"IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The "
"gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr ""
"Adjon meg IPv4 és/vagy IPv6 címet és előtagot (cím / előtag).  IPv4 esetén "
"használható a négytagú, pontokkal elválasztott hálózat maszk, vagy a CIDR "
"stílusú előtag. Az átjáró és a névkiszolgáló mezőkben érvényes IPv4 vagy "
"IPv6 címeknek kell szerepelni."

#: loader/net.c:1014
msgid "Manual TCP/IP Configuration"
msgstr "Kézi TCP/IP beállítás"

#: loader/net.c:1130 loader/net.c:1138
msgid "Missing Information"
msgstr "Hiányzó információ"

#: loader/net.c:1131
msgid ""
"You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr ""
"Meg kell adnia egy érvényes IPv4 címet, valamint vagy egy hálózat maszkot, "
"vagy egy CIDR-előtagot."

#: loader/net.c:1139
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr "Meg kell adni egy érvényes IPv6 címet és egy CIDR-előtagot."

#: loader/net.c:1797
#, c-format
msgid "Bad argument to kickstart network command: %s"
msgstr ""

#: loader/net.c:1821
#, c-format
msgid "Bad bootproto %s specified in network command"
msgstr "Network parancsban hibás a bootproto %s"

#: loader/net.c:1940
msgid "Seconds:"
msgstr "Másodpercek:"

#: loader/net.c:2159
msgid "Networking Device"
msgstr "Hálózat eszköz"

#: loader/net.c:2160
msgid ""
"You have multiple network devices on this system. Which would you like to "
"install through?"
msgstr ""
"Több hálózat eszköz van a rendszerben. Melyiken át szeretné elvégezni a "
"telepítést?"

#: loader/net.c:2164
msgid "Identify"
msgstr "Azonosítás"

#: loader/net.c:2173
msgid "You can identify the physical port for"
msgstr "Azonosíthat egy fizikai kaput "

#: loader/net.c:2175
msgid ""
"by flashing the LED lights for a number of seconds.  Enter a number between "
"1 and 30 to set the duration to flash the LED port lights."
msgstr ""
"eszközhöz a LED lámpácskát villogtatva egy megadott ideig.  Adjon meg egy "
"számot 1 és 30 között, hogy beállítsa a LED kapu fény villogás idejét."

#: loader/net.c:2182
msgid "Identify NIC"
msgstr "Hálózat csatoló azonosítása"

#: loader/net.c:2195
msgid "Invalid Duration"
msgstr "Érvénytelen időtartam"

#: loader/net.c:2196
msgid "You must enter the number of seconds as an integer between 1 and 30."
msgstr "Meg kell adnia a másodpercek számát 1 és 30 közötti egészként."

#: loader/net.c:2208
#, c-format
msgid "Flashing %s port lights for %d seconds."
msgstr "%s kapu fények villogtatása %d másodpercig."

#: loader/net.c:2439 loader/net.c:2443
#, c-format
msgid "Waiting for NetworkManager to configure %s.\n"
msgstr "Várok, hogy NetworkManager beállítsa %s eszközt.\n"

#: loader/nfsinstall.c:68
msgid "NFS server name:"
msgstr "NFS kiszolgáló név:"

#: loader/nfsinstall.c:72
#, c-format
msgid "%s directory:"
msgstr "%s mappa:"

#: loader/nfsinstall.c:76
msgid "NFS mount options (optional):"
msgstr ""

#: loader/nfsinstall.c:82
#, c-format
msgid ""
"Please enter the server and NFSv3 path to your %s installation image and "
"optionally additional NFS mount options."
msgstr ""

#: loader/nfsinstall.c:90
msgid "NFS Setup"
msgstr "NFS beállítás"

#: loader/nfsinstall.c:280
msgid "That directory could not be mounted from the server."
msgstr "Azt a mappát nem sikerült becsatolni"

#: loader/nfsinstall.c:292
#, c-format
msgid "That directory does not seem to contain a %s installation image."
msgstr "Úgy tűnik, az a mappa nem tartalmaz %s telepítő képet."

#: loader/nfsinstall.c:385
#, c-format
msgid "Bad argument to NFS kickstart method command: %s"
msgstr ""

#: loader/telnetd.c:89 loader/telnetd.c:128
msgid "Telnet"
msgstr "Telnet"

#: loader/telnetd.c:89
msgid "Waiting for telnet connection."
msgstr "Várakozás telnet kapcsolatra."

#: loader/telnetd.c:128
msgid "Running anaconda via telnet."
msgstr "Anaconda futtatása telneten át."

#: loader/urlinstall.c:138
#, c-format
msgid "Unable to retrieve %s."
msgstr ""

#: loader/urlinstall.c:222
msgid "Unable to retrieve the install image."
msgstr "Nem tudtam letölteni a telepítő képet."

#: loader/urlinstall.c:375
#, c-format
msgid "Bad argument to URL kickstart method command: %s"
msgstr ""

#: loader/urlinstall.c:386
msgid "Must supply a --url argument to Url kickstart method."
msgstr "Meg kell adni egy --url argumentumot az Url kickstart parancsnak."

#: loader/urlinstall.c:393
#, c-format
msgid "Unknown Url method %s"
msgstr "Ismeretlen Url parancs %s"

#: loader/urls.c:177 loader/urls.c:185
msgid "Retrieving"
msgstr "Letöltés"

#: loader/urls.c:271
#, c-format
msgid ""
"Please enter the URL containing the %s installation image on your server."
msgstr ""
"Kérem, adja meg az URL-t, ami %s telepítő képeket tartalmazza a kiszolgálón."

#: loader/urls.c:289
msgid "Enable HTTP proxy"
msgstr ""

#: loader/urls.c:303
msgid "Proxy URL"
msgstr ""

#: loader/urls.c:308
msgid "Username"
msgstr ""

#: loader/urls.c:313
msgid "Password"
msgstr ""

#: loader/urls.c:332
msgid "URL Setup"
msgstr "URL beállítás"

#: loader/urls.c:340
msgid "You must enter a URL."
msgstr "Meg kell adni egy URL-t."

#: loader/urls.c:346
msgid "URL must be either an ftp or http URL"
msgstr "Az URL vagy ftp vagy http URL kell legyen."

#: loader/urls.c:355
msgid "Proxy must be either an ftp or http URL"
msgstr ""

#: loader/windows.c:65
msgid "Loading SCSI driver"
msgstr "SCSI vezérlő betöltése"

#: loader/windows.c:66
#, c-format
msgid "Loading %s driver"
msgstr "%s vezérlő betöltése"

#: storage/__init__.py:101
msgid "Unknown Device"
msgstr "Ismeretlen eszköz"

#: storage/__init__.py:102
#, python-format
msgid ""
"The installation source given by device %s could not be found.  Please check "
"your parameters and try again."
msgstr ""
"A %s eszköz telepítő forrást nem találom.  Kérem ellenőrizze a "
"paramétereket, és próbálja újra."

#: storage/__init__.py:111
msgid "_Try again"
msgstr ""

#: storage/__init__.py:114
msgid "No disks found"
msgstr ""

#: storage/__init__.py:115
msgid "No usable disks have been found."
msgstr ""

#: storage/__init__.py:128
msgid "Installation cannot continue."
msgstr "A telepítés nem folytatható."

#: storage/__init__.py:129
msgid ""
"The storage configuration you have chosen has already been activated. You "
"can no longer return to the disk editing screen. Would you like to continue "
"with the installation process?"
msgstr ""
"A választott felosztás opciók már élnek. Már nem léphet vissza a lemez "
"szerkesztő képernyőre. Folytatni szeretné a telepítést?"

#: storage/__init__.py:157
msgid "Encrypt device?"
msgstr "Eszköz titkosítása?"

#: storage/__init__.py:158
msgid ""
"You specified block device encryption should be enabled, but you have not "
"supplied a passphrase. If you do not go back and provide a passphrase, block "
"device encryption will be disabled."
msgstr ""
"Megadta, hogy a blokk eszköz titkosítás engedélyezett legyen, de nem adott "
"meg jelszót. Ha nem ad meg egy jelszót, akkor a blokk eszköz titkosítása "
"letiltásra kerül."

#: storage/__init__.py:193
msgid ""
"\n"
"\n"
"WARNING:\n"
"You are using a GPT bootdisk on a non-EFI system. This may not work, "
"depending on your BIOS's support for booting from GPT disks."
msgstr ""

#: storage/__init__.py:212
msgid "Missing /boot Volume"
msgstr ""

#: storage/__init__.py:213
msgid ""
"This platform requires /boot on a dedicated partition or logical volume.  If "
"you do not want a /boot volume, you must place / on a dedicated non-LVM "
"partition."
msgstr ""

#: storage/__init__.py:220 storage/__init__.py:234
msgid "Go _back"
msgstr "_Vissza"

#: storage/__init__.py:227
msgid "Writing storage configuration to disk"
msgstr "Felosztás lemezre írása"

#: storage/__init__.py:228
#, python-format
msgid ""
"The partitioning options you have selected will now be written to disk.  Any "
"data on deleted or reformatted partitions will be lost.%s"
msgstr ""

#: storage/__init__.py:235
msgid "_Write changes to disk"
msgstr "Vált_ozások lemezre írása"

#: storage/__init__.py:254
msgid "Running..."
msgstr "Futtatás alatt..."

#: storage/__init__.py:255
msgid "Storing encryption keys"
msgstr "Titkosítási kulcsok mentése"

#: storage/__init__.py:271
#, python-format
msgid "Error storing an encryption key: %s\n"
msgstr "Hiba egy titkosítási kulcs letárolása közben: %s\n"

#: storage/__init__.py:422
msgid "Examining Devices"
msgstr ""

#: storage/__init__.py:423
msgid "Examining storage devices"
msgstr ""

#: storage/__init__.py:708
msgid "This partition is holding the data for the hard drive install."
msgstr "E rész tartalmazza a merevlemezes telepítéshez szükséges adatokat."

#: storage/__init__.py:713
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr "LDL DASD részt nem lehet törölni."

#: storage/__init__.py:719
#, python-format
msgid "This device is part of the RAID device %s."
msgstr "E rész a %s RAID eszközhöz tartozik."

#: storage/__init__.py:722
msgid "This device is part of a RAID device."
msgstr "E rész egy RAID eszközhöz tartozik."

#: storage/__init__.py:727
#, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr "Ez az eszköz „%s” LVM kötet csoporthoz tartozik."

#: storage/__init__.py:730
msgid "This device is part of a LVM volume group."
msgstr "E rész egy LVM kötetcsoporthoz tartozik."

#: storage/__init__.py:746
msgid ""
"This device is an extended partition which contains logical partitions that "
"cannot be deleted:\n"
"\n"
msgstr ""
"Ez az eszköz olyan kiterjesztett rész, ami nem törölhető logikai részeket "
"tartalmaz:\n"
"\n"

#: storage/__init__.py:1032
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr "Nem adta meg a gyökér részt (/), ami nélkül %s nem telepíthető."

#: storage/__init__.py:1037
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""
"A gyökér rész mérete kisebb 250 MB-nál, ami általában nem elég %s "
"telepítéséhez."

#: storage/__init__.py:1043
#, python-format
msgid ""
"Your / partition is less than %(min)s MB which is lower than recommended for "
"a normal %(productName)s install."
msgstr ""

#: storage/__init__.py:1050
#, python-format
msgid "Placing the root partition on an XFS filesystem is not supported in %s."
msgstr ""

#: storage/__init__.py:1058
#, python-format
msgid ""
"Your / partition does not match the the live image you are installing from.  "
"It must be formatted as %s."
msgstr ""
"A / rész nem egyezik meg az élő képpel, ahonnan telepít.  %s fájlrendszert "
"kell kialakítani."

#: storage/__init__.py:1065
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s megabytes which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""

#: storage/__init__.py:1075
#, python-format
msgid ""
"Your %s partition is too %s for %s formatting (allowable size is %d MB to %d "
"MB)"
msgstr ""

#: storage/__init__.py:1102
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""
"Telepítés USB eszközre.  Ez a megoldás működésképtelen rendszert is "
"eredményezhet."

#: storage/__init__.py:1105
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""
"Telepítés FireWire eszközre.  Ez a megoldás működésképtelen rendszert is "
"eredményezhet."

#: storage/__init__.py:1112
msgid ""
"You have not specified a swap partition.  Due to the amount of memory "
"present, a swap partition is required to complete installation."
msgstr ""
"Nem adott meg lapozó részt.  A memória mérete miatt, lapozó rész kell a "
"telepítés befejezéséhez."

#: storage/__init__.py:1117
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""
"Nem adott meg lapozó részt.  Erre nincs mindig szükség, de a legtöbb esetben "
"jelentősen növelheti a teljesítményt."

#: storage/__init__.py:1123
msgid ""
"At least one of your swap devices does not have a UUID, which is common in "
"swap space created using older versions of mkswap. These devices will be "
"referred to by device path in /etc/fstab, which is not ideal since device "
"paths can change under a variety of circumstances. "
msgstr ""

#: storage/__init__.py:1133
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""
"Ez a csatoló pont érvénytelen.  %s mappának a / fájlrendszerben kell lennie."

#: storage/__init__.py:1137
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "%s csatoló pontnak linuxos fájlrendszerben kell lennie."

#: storage/__init__.py:1148
msgid "No Drives Found"
msgstr "Nincs tároló eszköz"

#: storage/__init__.py:1149
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems. Please check your hardware for the cause of this problem."
msgstr ""
"Hiba történt – nincs érvényes eszköz, amin új fájlrendszert lehetne "
"kialakítani. Kérem ellenőrizze a hardvert, hogy kiderítse a probléma okát."

#: storage/__init__.py:1443 storage/__init__.py:1452
msgid "Dirty File Systems"
msgstr "Rendezetlen fájlrendszerek"

#: storage/__init__.py:1444
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""
"A következő fájlrendszereket nem tisztán csatolták ki.  Indítsa el a "
"rendszert, hagyja, hogy a fájlrendszerek ellenőrzése megtörténjen, majd "
"tisztán állítsa le a rendszert a frissítés  megkezdése előtt.\n"
"%s"

#: storage/__init__.py:1453
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""
"A rendszer következő részeit nem tisztán csatolták ki.  Biztosan be szeretné "
"csatolni őket?\n"
"%s"

#: storage/__init__.py:1938
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"is an old-style Linux swap partition.  If you want to use this device for "
"swap space, you must reformat as a new-style Linux swap partition."
msgstr ""
"A lapozó eszköz:\n"
"\n"
"     %s\n"
"\n"
"egy régi típusú Linux lapozó partíció. Ha ezt az eszközt kívánja használni "
"lapozóként, újra kell formáznia egy új típusúra."

#: storage/__init__.py:1949
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. To perform an upgrade, please shut "
"down your system rather than hibernating it."
msgstr ""
"     %s\n"
"\n"
"lapozó eszközt az /etc/fstab fájl szerint jelenleg szoftveres "
"felfüggesztéshez használják, ami annyit jelent, hogy a rendszer hibernált "
"állapotban van. A frissítés elvégzéséhez inkább állítsa le a rendszert, ne "
"hibernálja."

#: storage/__init__.py:1957
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap devices."
msgstr ""
"     %s\n"
"\n"
"lapozó eszközt az /etc/fstab fájl szerint jelenleg szoftveres "
"felfüggesztéshez használják, ami annyit jelent, hogy a rendszer hibernált "
"állapotban van. Ha új telepítést végez, győződjön meg arról, hogy a telepítő "
"újra alakítja az összes lapozó területet."

#: storage/__init__.py:1969
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"does not contain a supported swap volume.  In order to continue "
"installation, you will need to format the device or skip it."
msgstr ""

#: storage/__init__.py:1980
#, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap "
"device.\n"
"\n"
"Press OK to exit the installer"
msgstr ""

#: storage/__init__.py:1987
#, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s\n"
"\n"
"This most likely means this swap device has not been initialized.\n"
"\n"
"Press OK to exit the installer."
msgstr ""

#: storage/__init__.py:2050 storage/__init__.py:2062
msgid "Invalid mount point"
msgstr "Érvénytelen becsatoló pont"

#: storage/__init__.py:2051
#, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"%s előkészítése közben hiba történt.  Az elérési út néhány eleme nem mappa. "
"Ez végzetes hiba. A telepítés nem folytatható.\n"
"\n"
"Nyomjon <Enter>-t, hogy kilépjen a telepítőből."

#: storage/__init__.py:2063
#, python-format
msgid ""
"An error occurred when trying to create %(mountpoint)s: %(msg)s.  This is a "
"fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: storage/__init__.py:2079 storage/__init__.py:2103
msgid "Unable to mount filesystem"
msgstr "Nem lehet fájlrendszert becsatolni"

#: storage/__init__.py:2080
#, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s.  You may "
"continue installation, but there may be problems."
msgstr ""

#: storage/__init__.py:2104
#, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s: %(msg)s. This "
"is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: storage/dasd.py:142
msgid "Formatting DASD Device"
msgid_plural "Formatting DASD Devices"
msgstr[0] ""
msgstr[1] ""

#: storage/dasd.py:143
#, python-format
msgid "Preparing %d DASD device for use with Linux..."
msgid_plural "Preparing %d DASD devices for use with Linux..."
msgstr[0] ""
msgstr[1] ""

#: storage/devicelibs/lvm.py:323
#, python-format
msgid "vginfo failed for %s"
msgstr "vginfo sikertelen %s eszközre"

#: storage/devicelibs/lvm.py:352
#, python-format
msgid "lvs failed for %s"
msgstr "lvs sikertelen %s eszközre"

#: storage/devices.py:823 storage/devices.py:1547 storage/devices.py:2616
msgid "large"
msgstr ""

#: storage/devices.py:825 storage/devices.py:1553 storage/devices.py:2622
msgid "small"
msgstr ""

#: storage/devices.py:1323 storage/devices.py:2084 storage/devices.py:2549
#: storage/devices.py:3132 storage/devices.py:3619
msgid "Creating"
msgstr ""

#: storage/devices.py:1324 storage/devices.py:2085 storage/devices.py:2550
#: storage/devices.py:3133
#, python-format
msgid "Creating device %s"
msgstr ""

#: storage/devices.py:2665
#, python-format
msgid "A RAID%d set requires atleast %d members"
msgstr ""

#: storage/devices.py:3620
#, python-format
msgid "Creating file %s"
msgstr ""

#: storage/devicetree.py:102
msgid "Confirm"
msgstr "Megerősítés"

#: storage/devicetree.py:103
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during "
"installation."
msgstr ""
"Biztosan ki akarja hagyni a jelszó megadását %s eszközhöz?\n"
"\n"
"Ha átugorja e lépést, az eszköz tartalma elérhetetlen lesz a telepítés "
"közben."

#: storage/devicetree.py:1985 storage/devicetree.py:2033
msgid "This partition is part of an inconsistent LVM Volume Group."
msgstr "E rész egy ellentmondásos LVM kötetcsoporthoz tartozik."

#: storage/fcoe.py:71 storage/fcoe.py:72
msgid "Connecting to FCoE SAN"
msgstr ""

#: storage/fcoe.py:116
msgid "FCoE not available"
msgstr ""

#: storage/formats/fs.py:102
msgid "filesystem configuration missing a type"
msgstr "fájlrendszer beállításból hiányzik a típus"

#: storage/formats/fs.py:354 storage/formats/luks.py:199
#: storage/formats/lvmpv.py:103 storage/formats/swap.py:163
msgid "Formatting"
msgstr "Formázás"

#: storage/formats/fs.py:355
#, python-format
msgid "Creating %s filesystem on %s"
msgstr ""

#: storage/formats/fs.py:454
msgid "Resizing"
msgstr "Átméretezés"

#: storage/formats/fs.py:455
#, python-format
msgid "Resizing filesystem on %s"
msgstr "Fájlrendszer átméretezése %s partíción"

#: storage/formats/fs.py:490
#, python-format
msgid "Unknown return code: %d."
msgstr ""

#: storage/formats/fs.py:504
msgid "Checking"
msgstr "Ellenőrzés"

#: storage/formats/fs.py:505
#, python-format
msgid "Checking filesystem on %s"
msgstr "Fájlrendszer ellenőrzése %s partíción"

#: storage/formats/fs.py:522
#, python-format
msgid "%(type)s filesystem check failure on %(device)s: "
msgstr ""

#: storage/formats/fs.py:528
msgid ""
"Errors like this usually mean there is a problem with the filesystem that "
"will require user interaction to repair.  Before restarting installation, "
"reboot to rescue mode or another system that allows you to repair the "
"filesystem interactively.  Restart installation after you have corrected the "
"problems on the filesystem."
msgstr ""

#: storage/formats/fs.py:536 storage/partitioning.py:312
msgid "Unrecoverable Error"
msgstr "Helyrehozhatatlan hiba"

#: storage/formats/fs.py:917
msgid "File system errors left uncorrected."
msgstr ""

#: storage/formats/fs.py:918
msgid "Operational error."
msgstr ""

#: storage/formats/fs.py:919
msgid "Usage or syntax error."
msgstr ""

#: storage/formats/fs.py:920
msgid "e2fsck cancelled by user request."
msgstr ""

#: storage/formats/fs.py:921
msgid "Shared library error."
msgstr ""

#: storage/formats/fs.py:1079
msgid ""
"Recoverable errors have been detected or dosfsck has discovered an internal "
"inconsistency."
msgstr ""

#: storage/formats/fs.py:1081
msgid "Usage error."
msgstr ""

#: storage/formats/luks.py:48
msgid "Encrypted"
msgstr ""

#: storage/formats/luks.py:200
#, python-format
msgid "Encrypting %s"
msgstr ""

#: storage/formats/lvmpv.py:104 storage/formats/swap.py:164
#, python-format
msgid "Creating %s on %s"
msgstr ""

#: storage/iscsi.py:128
msgid "Unable to change iSCSI initiator name once set"
msgstr ""

#: storage/iscsi.py:130
msgid "Must provide an iSCSI initiator name"
msgstr ""

#: storage/iscsi.py:194 storage/iscsi.py:195
msgid "Scanning iSCSI nodes"
msgstr "iSCSI elemek lekérése"

#: storage/iscsi.py:246 storage/iscsi.py:247
msgid "Initializing iSCSI initiator"
msgstr "Az iSCSI inicializáló beállítása"

#: storage/iscsi.py:302
msgid "iSCSI not available"
msgstr "iSCSI nem érhető el"

#: storage/iscsi.py:304
msgid "No initiator name set"
msgstr "Nincs kezdeményező név beállítva"

#: storage/iscsi.py:343
msgid "Logging in to iSCSI node"
msgstr ""

#: storage/iscsi.py:344
#, python-format
msgid "Logging in to iSCSI node %s"
msgstr ""

#: storage/iscsi.py:378
msgid "No iSCSI nodes discovered"
msgstr "Nem találtam iSCSI elemeket"

#: storage/iscsi.py:400
msgid "No new iSCSI nodes discovered"
msgstr "Nem találtam új iSCSI elemeket"

#: storage/iscsi.py:403
msgid "Could not log in to any of the discovered nodes"
msgstr "Nem tudtam bejelentkezni egyik felfedezett elembe sem"

#: storage/partitioning.py:227
msgid ""
"Could not find enough free space for automatic partitioning.  Press 'OK' to "
"exit the installer."
msgstr ""
"Nem találtam elég szabad helyet az automata particionáláshoz. Nyomjon egy "
"'OK'-t a telepítőből való kilpéshez."

#: storage/partitioning.py:230
msgid ""
"Could not find enough free space for automatic partitioning, please use "
"another partitioning method."
msgstr ""
"Nem találtam elég szabad helyet automata particionáláshoz. Kérem használjon "
"más particionálási eljárást."

#: storage/partitioning.py:259
msgid "Warnings During Automatic Partitioning"
msgstr "Figyelmeztetések az automata particionálás során"

#: storage/partitioning.py:261
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""
"A következő figyelmeztetések érkeztek az automata particionálás közben:\n"
"\n"
"%s"

#: storage/partitioning.py:275 storage/partitioning.py:296
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"\n"
"\n"
"Nyomjon „OK”-t, hogy kilépjen a telepítőből."

#: storage/partitioning.py:277
#, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%(msg)s.%(extra)s"
msgstr ""

#: storage/partitioning.py:298
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""
"\n"
"\n"
"Nyomjon „OK” gombot, hogy más partícionálási lehetőséget válasszon."

#: storage/partitioning.py:300
msgid "Automatic Partitioning Errors"
msgstr "Az Automatikus Particionálás során fellépett hibák"

#: storage/partitioning.py:301
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %(extra)s"
msgstr ""

#: storage/zfcp.py:51
msgid "You have not specified a device number or the number is invalid"
msgstr "Nem adott meg eszközszámot, vagy a szám érvénytelen"

#: storage/zfcp.py:53
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr "Nem adott meg WWPN port nevet, vagy a név érvénytelen."

#: storage/zfcp.py:55
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr "Nem adott meg FCP LUN számot, vagy a szám érvénytelen."

#: storage/zfcp.py:133
#, python-format
msgid "zFCP device %s not found, not even in device ignore list."
msgstr "%s zFCP eszköz nem található, még a mellőzött eszközlistán sem."

#: storage/zfcp.py:143
#, python-format
msgid "Could not set zFCP device %(devnum)s online (%(e)s)."
msgstr ""

#: storage/zfcp.py:154
#, python-format
msgid "Could not add WWPN %(wwpn)s to zFCP device %(devnum)s (%(e)s)."
msgstr ""

#: storage/zfcp.py:161
#, python-format
msgid "WWPN %(wwpn)s not found at zFCP device %(devnum)s."
msgstr ""

#: storage/zfcp.py:176
#, python-format
msgid ""
"Could not add LUN %(fcplun)s to WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)"
"s)."
msgstr ""

#: storage/zfcp.py:182
#, python-format
msgid ""
"LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s already configured."
msgstr ""

#: storage/zfcp.py:194
#, python-format
msgid ""
"Could not read failed attribute of LUN %(fcplun)s at WWPN %(wwpn)s on zFCP "
"device %(devnum)s (%(e)s)."
msgstr ""

#: storage/zfcp.py:203
#, python-format
msgid ""
"Failed LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s removed "
"again."
msgstr ""

#: storage/zfcp.py:260
#, python-format
msgid ""
"Could not correctly delete SCSI device of zFCP %(devnum)s %(wwpn)s %(fcplun)"
"s (%(e)s)."
msgstr ""

#: storage/zfcp.py:269
#, python-format
msgid ""
"Could not remove LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s (%"
"(e)s)."
msgstr ""

#: storage/zfcp.py:287
#, python-format
msgid "Could not remove WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)s)."
msgstr ""

#: storage/zfcp.py:313
#, python-format
msgid "Could not set zFCP device %(devnum)s offline (%(e)s)."
msgstr ""

#: textw/complete_text.py:32
msgid "Reboot"
msgstr "Újraindítás"

#: textw/complete_text.py:34
msgid "<Enter> to exit"
msgstr "<Enter>, hogy kilépjen"

#: textw/complete_text.py:62
msgid "Complete"
msgstr "Kész"

#: textw/keyboard_text.py:48
msgid "Keyboard Selection"
msgstr "Billentyűzet kiválasztás"

#: textw/keyboard_text.py:49
msgid "Which model keyboard is attached to this computer?"
msgstr "Milyen típusú billentyűzet van a géphez kötve?"

#: textw/netconfig_text.py:35 textw/netconfig_text.py:42
msgid "Error With Data"
msgstr "Hibás adat"

#: textw/netconfig_text.py:36
#, python-format
msgid ""
"An error occurred converting the value entered for \"%(field)s\":\n"
"%(errmsg)s"
msgstr ""

#: textw/netconfig_text.py:43
#, python-format
msgid "A value is required for the field %s"
msgstr "Érték kell %s mezőnek"

#: textw/netconfig_text.py:126 textw/netconfig_text.py:152
msgid "Enable network interface"
msgstr "Hálózati csatoló engedélyezése"

#: textw/netconfig_text.py:129
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please select network interface to configure."
msgstr ""

#: textw/netconfig_text.py:155
#, python-format
msgid "Configure interface %s to be used during installation process."
msgstr ""

#: textw/netconfig_text.py:160
msgid "Enable IPv4 Support"
msgstr ""

#: textw/netconfig_text.py:167 textw/netconfig_text.py:207
msgid "Manual Configuration"
msgstr ""

#: textw/netconfig_text.py:175
msgid "IPv4 Address:"
msgstr "IPv4 cím:"

#: textw/netconfig_text.py:189 textw/netconfig_text.py:229
msgid "Nameserver:"
msgstr "Névkiszolgáló:"

#: textw/netconfig_text.py:197
msgid "Enable IPv6 Support"
msgstr ""

#: textw/netconfig_text.py:201
msgid "Automatic neighbor discovery"
msgstr ""

#: textw/netconfig_text.py:204
msgid "Dynamic IP Configuration (DHCPv6)"
msgstr ""

#: textw/netconfig_text.py:215
msgid "IPv6 Address:"
msgstr ""

#: textw/netconfig_text.py:270
msgid "Missing protocol"
msgstr ""

#: textw/netconfig_text.py:271
msgid "You must select at least one protocol version"
msgstr ""

#: textw/netconfig_text.py:279 textw/netconfig_text.py:282
msgid "IPv4 Address"
msgstr ""

#: textw/netconfig_text.py:286 textw/netconfig_text.py:291
msgid "IPv4 Prefix"
msgstr ""

#: textw/netconfig_text.py:290
msgid "IPv4 CIDR prefix must be between 0 and 32."
msgstr "IPv4 CIDR előtag 0 és 32 közé kell essen."

#: textw/netconfig_text.py:298
msgid "IPv4 Gateway"
msgstr ""

#: textw/netconfig_text.py:307
msgid "IPv4 Nameserver"
msgstr ""

#: textw/netconfig_text.py:315 textw/netconfig_text.py:318
msgid "IPv6 Address"
msgstr ""

#: textw/netconfig_text.py:322 textw/netconfig_text.py:327
msgid "IPv6 Prefix"
msgstr ""

#: textw/netconfig_text.py:326
msgid "IPv6 CIDR prefix must be between 0 and 128."
msgstr ""

#: textw/netconfig_text.py:334
msgid "IPv6 Gateway"
msgstr ""

#: textw/netconfig_text.py:342
msgid "IPv6 Nameserver"
msgstr ""

#: textw/netconfig_text.py:399
msgid "Configuring Network Interfaces"
msgstr "Hálózat csatolók beállítása"

#: textw/netconfig_text.py:404
#, python-format
msgid "There was an error configuring network device %s"
msgstr ""

#: textw/partition_text.py:62
msgid "Partitioning Type"
msgstr "Felosztás típusa"

#: textw/partition_text.py:64
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target."
msgstr ""
"A telepítés a merevlemez felosztását igényli.  Az alapértelmezett felosztás "
"megfelel a legtöbb használónak.  Válassza ki, mely helyet használjuk, és "
"mely eszközökre telepítsünk."

#: textw/partition_text.py:67
msgid "Use entire drive"
msgstr "Teljes eszköz használata"

#: textw/partition_text.py:68
msgid "Replace existing Linux system"
msgstr "Meglévő Linux rendszer lecserélése"

#: textw/partition_text.py:69
msgid "Use free space"
msgstr "Szabad hely használata"

#: textw/partition_text.py:84
msgid "Which drive(s) do you want to use for this installation?"
msgstr "Mely eszközöket szeretné használni a telepítéshez?"

#: textw/partition_text.py:99
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr "<Szóköz>,<+>,<->: kijelölés | <F2>: eszköz hozzáadás | <F12>: tovább"

#: textw/progress_text.py:46
msgid "Package Installation"
msgstr "Csomagok telepítése"

#: textw/timezone_text.py:74
msgid "In which time zone are you located?"
msgstr "Melyik időzónában tartózkodik?"

#: textw/timezone_text.py:92
msgid "System clock uses UTC"
msgstr "UTC rendszer óra"

#: textw/upgrade_bootloader_text.py:111 textw/upgrade_bootloader_text.py:122
msgid "Update boot loader configuration"
msgstr "Rendszerbetöltő beállítás frissítése"

#: textw/upgrade_bootloader_text.py:125
msgid "Skip boot loader updating"
msgstr "Rendszerbetöltő frissítés kihagyása"

#: textw/upgrade_text.py:41
#, python-format
msgid ""
"This release of %(productName)s supports the an updated file system, which "
"has several benefits over the file system traditionally shipped in %"
"(productName)s.  This installation program can migrate formatted partitions "
"without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""

#: textw/upgrade_text.py:128
msgid "Free Space"
msgstr "Szabad hely"

#: textw/upgrade_text.py:146
msgid "RAM detected (MB):"
msgstr "Érzékelt memória (MB):"

#: textw/upgrade_text.py:149
msgid "Suggested size (MB):"
msgstr "Javasolt méret (MB):"

#: textw/upgrade_text.py:152
msgid "Swap file size (MB):"
msgstr "Lapozó fájl méret (MB):"

#: textw/upgrade_text.py:160
msgid "Add Swap"
msgstr "Lapozó hozzáadása"

#: textw/upgrade_text.py:185
msgid "The value you entered is not a valid number."
msgstr "A megadott érték nem érvényes szám."

#: textw/upgrade_text.py:228
msgid "Reinstall System"
msgstr "Rendszer újratelepítése"

#: textw/upgrade_text.py:238
msgid "System to Upgrade"
msgstr "Frissitendő rendszer"

#: textw/upgrade_text.py:239
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to upgrade, or select 'Reinstall System' to freshly "
"install your system."
msgstr ""
"Legalább egy létező Linux-rendszer található a gépen.\n"
"\n"
"Válassza ki a felfejlesztendőt, vagy friss rendszer telepítéséhez válassza "
"„Rendszer újratelepítése” lehetőséget."

#: textw/userauth_text.py:30
msgid "Root Password"
msgstr "Rendszergazda jelszó"

#: textw/userauth_text.py:33
msgid ""
"Pick a root password. You must type it twice to ensure you know it and do "
"not make a typing mistake. "
msgstr ""
"Válasszon rendszergazda jelszót. A jelszót kétszer kell beírnia, hogy "
"biztosan tudja, és ne gépelje el."

#: textw/userauth_text.py:67
msgid "The root password must be at least 6 characters long."
msgstr "Rendszergazda jelszó legalább 6 jel hosszú kell legyen."

#: textw/welcome_text.py:29
#, python-format
msgid "%s"
msgstr "%s"

#: textw/welcome_text.py:30
#, python-format
msgid ""
"Welcome to %s!\n"
"\n"
msgstr ""
"Üdvözöli %s!\n"
"\n"

#: textw/zipl_text.py:35
msgid ""
"The z/IPL Boot Loader will be installed on your system after installation is "
"complete. You can now enter any additional kernel and chandev parameters "
"which your machine or your setup require."
msgstr ""
"A z/IPL rendszerindító lesz feltelepítve a telepítés végén. Most megadhatók "
"a gép illetve a konfiguráció működéséhez szükséges további rendszermag- és "
"chandev-paraméterek."

#: textw/zipl_text.py:67
msgid "z/IPL Configuration"
msgstr "z/IPL beállítás"

#: textw/zipl_text.py:75 textw/zipl_text.py:79
msgid "Chandev line "
msgstr "Chandev sor "

#: liveinst/liveinst.desktop.in.h:1
msgid "Install"
msgstr "Telepítés"

#: liveinst/liveinst.desktop.in.h:2
msgid "Install the live CD to your hard disk"
msgstr "Live CD telepítése a merevlemezre"

#: liveinst/liveinst.desktop.in.h:3
msgid "Install to Hard Drive"
msgstr "Telepítés merevlemezre"

#: ui/GroupSelector.glade.h:1
msgid ""
"Some packages associated with this group are not required to be installed "
"but may provide additional functionality.  Please choose the packages which "
"you would like to have installed."
msgstr ""
"E csoportban néhány csomagot nem szükséges telepíteni, de további "
"szolgáltatásokat tehetnek lehetővé.  Kérem válassza ki a telepíteni kívánt "
"csomagokat."

#: ui/GroupSelector.glade.h:2
msgid "_Deselect"
msgstr "Ki_húzás"

#: ui/GroupSelector.glade.h:3
msgid "_Deselect all optional packages"
msgstr "Minden választható csomag ki_húzása"

#: ui/GroupSelector.glade.h:4
msgid "_Optional packages"
msgstr "_Választható csomagok"

#: ui/GroupSelector.glade.h:5
msgid "_Select"
msgstr "_Bejelölés"

#: ui/GroupSelector.glade.h:6
msgid "_Select all optional packages"
msgstr "_Minden választható csomag megjelölése"

#: ui/GroupSelector.glade.h:7
msgid "dialog1"
msgstr "dialog1"

#: ui/account.glade.h:1
msgid "Confirm:"
msgstr "Megerősítés:"

#: ui/account.glade.h:2
msgid "Root Password:"
msgstr "Rendszergazda jelszó:"

#: ui/account.glade.h:3
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"A root számlát rendszer adminisztrációra használják.  Adja jelszót a root "
"használónak."

#: ui/adddrive.glade.h:1
msgid "Active network interfaces:"
msgstr ""

#: ui/adddrive.glade.h:2
msgid "Add _FCoE SAN"
msgstr "_FCoE SAN hozzáadása"

#: ui/adddrive.glade.h:3
msgid "Add _ZFCP LUN"
msgstr "_ZFCP LUN hozzáadása"

#: ui/adddrive.glade.h:4
msgid "Add _iSCSI target"
msgstr "_iSCSI cél hozzáadása"

#: ui/adddrive.glade.h:5
msgid "Advanced Storage Options"
msgstr "További tároló lehetőségek"

#: ui/adddrive.glade.h:6
msgid "C_onfigure Network"
msgstr ""

#: ui/adddrive.glade.h:7
msgid "How would you like to modify your drive configuration?"
msgstr "Hogyan szeretné módosítani az eszköz beállítást?"

#: ui/adddrive.glade.h:8
msgid "None"
msgstr ""

#: ui/adddrive.glade.h:9
msgid "_Add drive"
msgstr "_Vezérlő hozzáadása"

#: ui/adddrive.glade.h:10
msgid "_Bind targets to network interfaces"
msgstr ""

#: ui/addrepo.glade.h:1
msgid "<b>Repository _name:</b>"
msgstr "<b>Tároló _név:</b>"

#: ui/addrepo.glade.h:2
msgid "<b>Repository _type:</b>"
msgstr "<b>Tároló _típus:</b>"

#: ui/addrepo.glade.h:4
msgid "Configure _proxy"
msgstr "_Proxy beállítás"

#: ui/addrepo.glade.h:5
msgid ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Hard Drive"
msgstr ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Merevlemez"

#: ui/addrepo.glade.h:9
msgid "Opt_ions"
msgstr ""

#: ui/addrepo.glade.h:10
msgid ""
"Please provide the configuration information for this software repository."
msgstr "Kérem, adja meg beállítás információkat e szoftver tárolóhoz."

#: ui/addrepo.glade.h:11
msgid "Proxy U_RL (host:port)"
msgstr "Átjátszó URL (gazda:kapu)"

#: ui/addrepo.glade.h:12
msgid "Proxy pass_word"
msgstr "Proxy jel_szó"

#: ui/addrepo.glade.h:13
msgid "Proxy u_sername"
msgstr "Proxy _használó"

#: ui/addrepo.glade.h:14
msgid "Repository _URL"
msgstr "Tároló _URL"

#: ui/addrepo.glade.h:15
msgid "Select A Directory"
msgstr "Mappa választás"

#: ui/addrepo.glade.h:16
msgid "URL is a _mirror list"
msgstr "URL egy _tükör lista"

#: ui/addrepo.glade.h:17
msgid "_Directory"
msgstr "_Mappa"

#: ui/addrepo.glade.h:18
msgid "_Partition"
msgstr "_Rész"

#: ui/addrepo.glade.h:19
msgid "_Path"
msgstr "_Út"

#: ui/addrepo.glade.h:20
msgid "_Server"
msgstr "_Kiszolgáló"

#: ui/anaconda.glade.h:1
msgid "Reboo_t"
msgstr "Ú_jraindítás"

#: ui/anaconda.glade.h:4
msgid "_Next"
msgstr "_Tovább"

#: ui/autopart.glade.h:1
msgid "<b>Shrink partition _to size (in MB):</b>"
msgstr ""

#: ui/autopart.glade.h:2
msgid "Re_view and modify partitioning layout"
msgstr "Felosztás felül_vizsgálata és megváltoztatása"

#: ui/autopart.glade.h:3
msgid "Volume to Shrink"
msgstr "Csökkentendő kötet"

#: ui/autopart.glade.h:4
msgid ""
"Which partition would you like to shrink to make room for your installation?"
msgstr ""
"Melyik részt szeretné csökkenteni, hogy helyet készítsen a telepítésnek?"

#: ui/autopart.glade.h:5
msgid "Which type of installation would you like?"
msgstr ""

#: ui/autopart.glade.h:6
msgid "_Encrypt system"
msgstr "Rendszer _titkosítása"

#: ui/autopart.glade.h:7
msgid "_Shrink"
msgstr "Z_sugorít"

#: ui/blwhere.glade.h:1
msgid "/boot"
msgstr "/boot"

#: ui/blwhere.glade.h:2
msgid "BIOS Drive Order"
msgstr "BIOS eszköz sorrend"

#: ui/blwhere.glade.h:3
msgid "Boot loader device"
msgstr "Rendszerindító eszköz"

#: ui/blwhere.glade.h:4
msgid "First BIOS drive:"
msgstr "Első BIOS eszköz:"

#: ui/blwhere.glade.h:5
msgid "Fourth BIOS drive:"
msgstr "Negyedik BIOS eszköz:"

#: ui/blwhere.glade.h:6
msgid "MBR"
msgstr "MBR"

#: ui/blwhere.glade.h:7
msgid "Second BIOS drive:"
msgstr "Második BIOS eszköz:"

#: ui/blwhere.glade.h:8
msgid "Third BIOS drive:"
msgstr "Harmadik BIOS eszköz:"

#: ui/blwhere.glade.h:9
msgid "Where would you like to install the boot loader for your system?"
msgstr "Hova szeretné telepíteni a rendszerindítót?"

#: ui/cleardisks.glade.h:1
msgid "<b>Data Storage Devices</b> (to be mounted only)"
msgstr ""

#: ui/cleardisks.glade.h:2
msgid "<b>Install Target Devices</b>"
msgstr ""

#: ui/cleardisks.glade.h:4
msgid ""
"Below are the storage devices you've selected to be a part of this "
"installation.  Please indicate using the arrows below which devices you'd "
"like to use as data drives (these will not be formatted, only mounted) and "
"which devices you'd like to use as system drives (these may be formatted).  "
"Please also indicate which system drive will have the bootloader installed."
msgstr ""

#: ui/create-storage.glade.h:1
msgid ""
"<span size=\"small\" color=\"gray\">Create a RAID formated partition</span>"
msgstr ""

#: ui/create-storage.glade.h:2
msgid ""
"<span size=\"small\" color=\"gray\">Create a logical volume on selected "
"volume group</span>"
msgstr ""

#: ui/create-storage.glade.h:3
msgid ""
"<span size=\"small\" color=\"gray\">Create an LVM formated partition</span>"
msgstr ""

#: ui/create-storage.glade.h:4
msgid ""
"<span size=\"small\" color=\"gray\">General purpose partition creation</span>"
msgstr ""

#: ui/create-storage.glade.h:5
msgid ""
"<span size=\"small\" color=\"gray\">Requires at least 1 free LVM formated "
"partition</span>"
msgstr ""

#: ui/create-storage.glade.h:6
msgid ""
"<span size=\"small\" color=\"gray\">Requires at least 2 free RAID formated "
"partitions</span>"
msgstr ""

#: ui/create-storage.glade.h:7
msgid "C_reate"
msgstr ""

#: ui/create-storage.glade.h:8
msgid "Create LVM"
msgstr ""

#: ui/create-storage.glade.h:9
msgid "Create Partition"
msgstr ""

#: ui/create-storage.glade.h:10
msgid "Create Software RAID"
msgstr ""

#: ui/create-storage.glade.h:11
msgid "Create Storage"
msgstr ""

#: ui/create-storage.glade.h:12
msgid "LVM Logical Volume"
msgstr ""

#: ui/create-storage.glade.h:13
msgid "LVM Physical Volume"
msgstr ""

#: ui/create-storage.glade.h:14
msgid "LVM Volume Group"
msgstr ""

#: ui/create-storage.glade.h:16
msgid "RAID Partition"
msgstr ""

#: ui/create-storage.glade.h:17
msgid "Standard Partition"
msgstr ""

#: ui/detailed-dialog.glade.h:1
msgid "Info"
msgstr "Infó"

#: ui/detailed-dialog.glade.h:2
msgid "_Details"
msgstr "_Részletek"

#: ui/device-options.glade.h:1
msgid "Device Options"
msgstr ""

#: ui/device-options.glade.h:2
msgid ""
"If enabled the multipath devices are named following the /dev/mapper/"
"mpath<letter> format instead of /dev/mapper/<wwid>. Note that the user "
"friendly names are assigned by multipath tools and are not guaranteed to be "
"the same between installs."
msgstr ""

#: ui/device-options.glade.h:3
msgid "Use _friendly names for multipath devices"
msgstr ""

#: ui/fcoe-config.glade.h:1
msgid "Configure FCoE Parameters"
msgstr "FCoE paraméterek beállítása"

#: ui/fcoe-config.glade.h:2
msgid "NIC:"
msgstr ""

#: ui/fcoe-config.glade.h:3
msgid ""
"Please select the network interface which is connected to\n"
"your FCoE switch."
msgstr ""
"Kérem válassza ki a hálózati csatlakozót ami kapcsolódik\n"
"az ö FCoE switch-hez."

#: ui/fcoe-config.glade.h:5
msgid "Use DCB"
msgstr ""

#: ui/fcoe-config.glade.h:6
msgid "Use auto vlan"
msgstr ""

#: ui/fcoe-config.glade.h:7
msgid "_Add FCoE Disk(s)"
msgstr "FCoE lemez(ek) hozzá_adása"

#: ui/filter.glade.h:2
#, no-c-format
msgid "<b>%s devices (%s) selected</b> out of %s devices (%s) total."
msgstr ""

#: ui/filter.glade.h:3
msgid ""
"<b>Tip:</b> Selecting a drive on this screen does not necessarily mean it "
"will be wiped by the installation process.  Also, note that post-"
"installation you may mount drives you did not select here by modifying your /"
"etc/fstab file."
msgstr ""

#: ui/filter.glade.h:4
msgid "Add Advanced Target"
msgstr ""

#: ui/filter.glade.h:5
msgid "Basic Devices"
msgstr ""

#: ui/filter.glade.h:6
msgid "Filter By:"
msgstr ""

#: ui/filter.glade.h:7
msgid "Firmware RAID"
msgstr ""

#: ui/filter.glade.h:8
msgid "Identifier:"
msgstr ""

#: ui/filter.glade.h:9
msgid ""
"Interconnect\n"
"Vendor\n"
"Identifier"
msgstr ""

#: ui/filter.glade.h:12
msgid "LUN:"
msgstr ""

#: ui/filter.glade.h:13
msgid "Multipath Devices"
msgstr ""

#: ui/filter.glade.h:14
msgid "Other SAN Devices"
msgstr ""

#: ui/filter.glade.h:15
msgid ""
"Please select the drives you'd like to install the operating system on, as "
"well as any drives you'd like to automatically mount to your system, below:"
msgstr ""

#: ui/filter.glade.h:16
msgid ""
"Port / Target / LUN\n"
"Target Identifier"
msgstr ""

#: ui/filter.glade.h:18
msgid "Port:"
msgstr ""

#: ui/filter.glade.h:19
msgid "Search"
msgstr ""

#: ui/filter.glade.h:20
msgid "Search By:"
msgstr ""

#: ui/filter.glade.h:21
msgid "Search Results:"
msgstr ""

#: ui/filter.glade.h:22
msgid "Show Identifiers that Include:"
msgstr ""

#: ui/filter.glade.h:23
msgid "Show Only Devices From:"
msgstr ""

#: ui/filter.glade.h:24
msgid "Show Only Devices Using:"
msgstr ""

#: ui/filter.glade.h:25
msgid "Target:"
msgstr ""

#: ui/filter.glade.h:26
msgid "_Device Options"
msgstr ""

#: ui/iscsi-dialogs.glade.h:1
msgid "CHAP Password:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:2
msgid "CHAP Username:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:3
msgid "Check the nodes you wish to log into:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:4
msgid "Reason:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:5
msgid "Retry _Login"
msgstr ""

#: ui/iscsi-dialogs.glade.h:6
msgid "Reverse CHAP Password:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:7
msgid "Reverse CHAP Username:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:8
msgid "Start _Discovery"
msgstr ""

#: ui/iscsi-dialogs.glade.h:9
msgid "Successfully logged in and attached the following nodes:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:10
msgid "Target IP Address:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:11
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""
"Az iSCSI lemezek használatához meg kell adnia az iSCSI cél eszköz címét és "
"az iSCSI kezdeményező nevét, amit a számítógépen beállított."

#: ui/iscsi-dialogs.glade.h:12
msgid ""
"What kind of iSCSI <b>discovery authentication</b> do you wish to perform:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:13
msgid "What kind of iSCSI <b>login authentication</b> do you wish to perform:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:14
msgid "_Cancel"
msgstr ""

#: ui/iscsi-dialogs.glade.h:15
msgid "_Login"
msgstr ""

#: ui/iscsi-dialogs.glade.h:16
msgid "_OK"
msgstr ""

#: ui/iscsi-dialogs.glade.h:17
msgid "iSCSI Discovered Nodes"
msgstr ""

#: ui/iscsi-dialogs.glade.h:18
msgid "iSCSI Discovery Details"
msgstr ""

#: ui/iscsi-dialogs.glade.h:19
msgid "iSCSI Initiator Name:"
msgstr ""

#: ui/iscsi-dialogs.glade.h:20
msgid "iSCSI Login Results"
msgstr ""

#: ui/iscsi-dialogs.glade.h:21
msgid "iSCSI Nodes Login"
msgstr ""

#: ui/iscsi-dialogs.glade.h:22
msgid "iSCSI login has failed for the following nodes:"
msgstr ""

#: ui/lukspassphrase.glade.h:1
msgid ""
"Also add this passphrase to all existing encrypted devices to streamline the "
"boot process"
msgstr ""
"E jelszó használata minden meglévő titkosított eszközhöz, hogy meggyorsítsuk "
"a betöltés folyamatát"

#: ui/lukspassphrase.glade.h:2
msgid ""
"Choose a passphrase for this encrypted partition. You will be prompted for "
"the passphrase during system boot."
msgstr ""
"Válasszon jelszót e titkosított résznek. A rendszerindító induláskor kérni "
"fogja ezt a jelszót."

#: ui/lukspassphrase.glade.h:3
msgid "Confirm passphrase:"
msgstr "Jelszó megerősítése:"

#: ui/lukspassphrase.glade.h:4
msgid "Enter passphrase for encrypted partition"
msgstr "Adja meg a titkosított rész jelszavát"

#: ui/lukspassphrase.glade.h:5
msgid "Enter passphrase:"
msgstr "Jelszó megadása:"

#: ui/network.glade.h:1
msgid "Hostname:"
msgstr "Gazdanév:"

#: ui/network.glade.h:2
msgid ""
"Please name this computer.  The hostname identifies the computer on a "
"network."
msgstr ""
"Kérem nevezze el e számítógépet.  Hálózatban a gazda név azonosítja a "
"számítógépet."

#: ui/network.glade.h:3
msgid "_Configure Network"
msgstr ""

#: ui/tasksel.glade.h:1
msgid "Customize _later"
msgstr "Testreszabás _később"

#: ui/tasksel.glade.h:2
msgid ""
"Please select any additional repositories that you want to use for software "
"installation."
msgstr ""
"Kérem válasszon ki minden további tárolót, amit használni akar a szoftver "
"telepítéséhez."

#: ui/tasksel.glade.h:4
#, no-c-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage.  What additional tasks would you like your system to "
"support?"
msgstr ""
"%s alapértelmezett telepítése tartalmaz egy általános Internet használathoz "
"készült szoftver csoportot.  Milyen további feladatokat szeretne, hogy a "
"rendszer támogassa?"

#: ui/tasksel.glade.h:5
msgid ""
"You can further customize the software selection now, or after install via "
"the software management application."
msgstr ""
"Még tovább testreszabhatja a szoftver válogatást most vagy később a "
"telepítés után a szoftverkezelő alkalmazással."

#: ui/tasksel.glade.h:6
msgid "_Add additional software repositories"
msgstr "További szoftver tárolók _hozzáadása"

#: ui/tasksel.glade.h:7
msgid "_Customize now"
msgstr "_Testreszabás most"

#: ui/tasksel.glade.h:8
msgid "_Modify repository"
msgstr "_Tároló megváltoztatása"

#: ui/zfcp-config.glade.h:1
msgid "<b>Device number:</b>"
msgstr "<b>Eszköz szám:</b>"

#: ui/zfcp-config.glade.h:2
msgid "<b>FCP LUN:</b>"
msgstr "<b>FCP LUN:</b>"

#: ui/zfcp-config.glade.h:3
msgid "<b>WWPN:</b>"
msgstr "<b>WWPN:</b>"

#: ui/zfcp-config.glade.h:4
msgid "Add FCP device"
msgstr "FCP eszköz hozzáadása"

#: ui/zfcp-config.glade.h:5
msgid ""
"zSeries machines can access industry-standard SCSI devices via Fibre Channel "
"(FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port "
"Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr ""
"A zSeries számítógépek hozzá tudnak férni Fibre Channel-ön (FCP) keresztül a "
"szabványos SCSI-eszközökhöz. A következőket kell megadni minden eszközhöz: "
"16 bites eszközszám, 64 bites egyedi portnév (World Wide Port Name, WWPN), "
"64 bites FCP LUN."

#: ui/reinitialize-dialog.glade.h:1
msgid ""
"<span size=\"large\" weight=\"bold\">The storage device below may contain "
"data.</span>"
msgstr ""

#: ui/reinitialize-dialog.glade.h:3
msgid ""
"We could not detect partitions or filesystems on this device.\n"
"\n"
"This could be because the device is <b>blank</b>, <b>unpartitioned</b>, or "
"<b>virtual</b>. If not, there may be data on the device that can not be "
"recovered if you use it in this installation. We can remove the device from "
"this installation to protect the data.\n"
"\n"
"Are you sure this device does not contain valuable data?"
msgstr ""

#: ui/reinitialize-dialog.glade.h:8
msgid ""
"_Apply my choice to all devices with undetected partitions or filesystems"
msgstr ""

#: ui/reinitialize-dialog.glade.h:9
msgid "_No, keep any data"
msgstr ""

#: ui/reinitialize-dialog.glade.h:10
msgid "_Yes, discard any data"
msgstr ""

#: lang-table.h:1
msgid "Afrikaans"
msgstr "Afrikai"

#: lang-table.h:2
msgid "Arabic"
msgstr "Arab"

#: lang-table.h:3
msgid "Assamese"
msgstr "Assami"

#: lang-table.h:4
msgid "Bengali"
msgstr "Bengáli"

#: lang-table.h:5
msgid "Bengali(India)"
msgstr "Bengáli (India)"

#: lang-table.h:6
msgid "Bulgarian"
msgstr "Bulgár"

#: lang-table.h:7
msgid "Catalan"
msgstr "Katalán"

#: lang-table.h:8
msgid "Chinese(Simplified)"
msgstr "Chinese (Egyszerű)"

#: lang-table.h:9
msgid "Chinese(Traditional)"
msgstr "Kínai (Hagyományos)"

#: lang-table.h:10
msgid "Croatian"
msgstr "Horvát"

#: lang-table.h:11
msgid "Czech"
msgstr "Cseh"

#: lang-table.h:12
msgid "Danish"
msgstr "Dán"

#: lang-table.h:13
msgid "Dutch"
msgstr "Holland"

#: lang-table.h:14
msgid "English"
msgstr "Angol"

#: lang-table.h:15
msgid "Estonian"
msgstr "Észt"

#: lang-table.h:16
msgid "Finnish"
msgstr "Finn"

#: lang-table.h:17
msgid "French"
msgstr "Francia"

#: lang-table.h:18
msgid "German"
msgstr "Német"

#: lang-table.h:19
msgid "Greek"
msgstr "Görög"

#: lang-table.h:20
msgid "Gujarati"
msgstr "Gujarati"

#: lang-table.h:21
msgid "Hebrew"
msgstr "Héber"

#: lang-table.h:22
msgid "Hindi"
msgstr "Hindi"

#: lang-table.h:23
msgid "Hungarian"
msgstr "Magyar"

#: lang-table.h:24
msgid "Icelandic"
msgstr "Izlandi"

#: lang-table.h:25
msgid "Iloko"
msgstr "Iloko"

#: lang-table.h:26
msgid "Indonesian"
msgstr "Indonéz"

#: lang-table.h:27
msgid "Italian"
msgstr "Olasz"

#: lang-table.h:28
msgid "Japanese"
msgstr "Japán"

#: lang-table.h:29
msgid "Kannada"
msgstr "Kannada"

#: lang-table.h:30
msgid "Korean"
msgstr "Koreai"

#: lang-table.h:31
msgid "Macedonian"
msgstr "Macedón"

#: lang-table.h:32
msgid "Maithili"
msgstr "Maithili"

#: lang-table.h:33
msgid "Malay"
msgstr "Maláj"

#: lang-table.h:34
msgid "Malayalam"
msgstr "Malayalam-i"

#: lang-table.h:35
msgid "Marathi"
msgstr "Maráthi"

#: lang-table.h:36
msgid "Nepali"
msgstr "Nepál"

#: lang-table.h:37
msgid "Norwegian(Bokmål)"
msgstr "Norvég (Bokmål)"

#: lang-table.h:38
msgid "Northern Sotho"
msgstr "Északi Sotho"

#: lang-table.h:39
msgid "Oriya"
msgstr "Oriya"

#: lang-table.h:40
msgid "Persian"
msgstr "Perzsa"

#: lang-table.h:41
msgid "Polish"
msgstr "Lengyel"

#: lang-table.h:42
msgid "Portuguese"
msgstr "Portugál"

#: lang-table.h:43
msgid "Portuguese(Brazilian)"
msgstr "Portugál (Brazil)"

#: lang-table.h:44
msgid "Punjabi"
msgstr "India"

#: lang-table.h:45
msgid "Romanian"
msgstr "Román"

#: lang-table.h:46
msgid "Russian"
msgstr "Orosz"

#: lang-table.h:47
msgid "Serbian"
msgstr "Szerb"

#: lang-table.h:48
msgid "Serbian(Latin)"
msgstr "Szerb (Latin)"

#: lang-table.h:49
msgid "Sinhala"
msgstr "Sinhala"

#: lang-table.h:50
msgid "Slovak"
msgstr "Szlovák"

#: lang-table.h:51
msgid "Slovenian"
msgstr "Szlovén"

#: lang-table.h:52
msgid "Spanish"
msgstr "Spanyol"

#: lang-table.h:53
msgid "Swedish"
msgstr "Svéd"

#: lang-table.h:54
msgid "Tajik"
msgstr "Thai"

#: lang-table.h:55
msgid "Tamil"
msgstr "Tamil"

#: lang-table.h:56
msgid "Telugu"
msgstr "Telugu"

#: lang-table.h:57
msgid "Turkish"
msgstr "Török"

#: lang-table.h:58
msgid "Ukrainian"
msgstr "Ukrán"

#: lang-table.h:59
msgid "Vietnamese"
msgstr "Vietnam"

#: lang-table.h:60
msgid "Welsh"
msgstr "Welsh"

#: lang-table.h:61
msgid "Zulu"
msgstr "Zulu"
